<article itemscope="" itemtype=http://schema.org/Article data-itemscope=root> <h1 itemprop=headline>Replication Study: Transcriptional amplification in tumor cells with elevated c-Myc</h1> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Replication%20Study:%20Transcriptional%20amplification%20in%20tumor%20cells%20with%20elevated%20c-Myc"> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Michelle Lewis"><span data-itemprop=givenNames><span itemprop=givenName>L</span><span itemprop=givenName>Michelle</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lewis</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-1>1</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Meredith C Edwards"><span data-itemprop=givenNames><span itemprop=givenName>Meredith</span><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Edwards</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-1>1</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Zachary R Meyers"><span data-itemprop=givenNames><span itemprop=givenName>Zachary</span><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Meyers</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-1>1</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Conover Talbot"><span data-itemprop=givenNames><span itemprop=givenName>C</span><span itemprop=givenName>Conover</span></span><span data-itemprop=familyNames><span itemprop=familyName>Talbot</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-2>2</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Haiping Hao"><span data-itemprop=givenNames><span itemprop=givenName>Haiping</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hao</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-2>2</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="David Blum"><span data-itemprop=givenNames><span itemprop=givenName>David</span></span><span data-itemprop=familyNames><span itemprop=familyName>Blum</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-1>1</a></span></li> <li itemscope="" itemtype=http://schema.org/Organization itemprop=author><span itemprop=name>Reproducibility Project: Cancer Biology</span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Elizabeth Iorns"><span data-itemprop=givenNames><span itemprop=givenName>Elizabeth</span></span><span data-itemprop=familyNames><span itemprop=familyName>Iorns</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-3>3</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Rachel Tsui"><span data-itemprop=givenNames><span itemprop=givenName>Rachel</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tsui</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-3>3</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Alexandria Denis"><span data-itemprop=givenNames><span itemprop=givenName>Alexandria</span></span><span data-itemprop=familyNames><span itemprop=familyName>Denis</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-4>4</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Nicole Perfito"><span data-itemprop=givenNames><span itemprop=givenName>Nicole</span></span><span data-itemprop=familyNames><span itemprop=familyName>Perfito</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-3>3</a></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Timothy M Errington"><span data-itemprop=givenNames><span itemprop=givenName>Timothy</span><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Errington</span></span><span data-itemprop=affiliations><a itemprop=affiliation href=#author-organization-4>4</a></span></li> </ol> <ol data-itemprop=affiliations> <li itemscope="" itemtype=http://schema.org/Organization itemid=#author-organization-1 id=author-organization-1><span itemprop=name>University of Georgia, Bioexpression and Fermentation Facility</span> <address itemscope="" itemtype=http://schema.org/PostalAddress itemprop=address><span itemprop=addressLocality>Georgia</span><span itemprop=addressCountry>United States</span></address> </li> <li itemscope="" itemtype=http://schema.org/Organization itemid=#author-organization-2 id=author-organization-2><span itemprop=name>Johns Hopkins University, Deep Sequencing and Microarray Core Facility</span> <address itemscope="" itemtype=http://schema.org/PostalAddress itemprop=address><span itemprop=addressLocality>Maryland</span><span itemprop=addressCountry>United States</span></address> </li> <li itemscope="" itemtype=http://schema.org/Organization itemid=#author-organization-3 id=author-organization-3><span itemprop=name>Science Exchange</span> <address itemscope="" itemtype=http://schema.org/PostalAddress itemprop=address><span itemprop=addressLocality>Palo Alto</span><span itemprop=addressCountry>United States</span></address> </li> <li itemscope="" itemtype=http://schema.org/Organization itemid=#author-organization-4 id=author-organization-4><span itemprop=name>Center for Open Science</span> <address itemscope="" itemtype=http://schema.org/PostalAddress itemprop=address><span itemprop=addressLocality>Charlottesville</span><span itemprop=addressCountry>United States</span></address> </li> </ol><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span><time itemprop=datePublished datetime=2018-01-09>2018-01-09</time> <ul data-itemprop=identifiers> <li itemscope="" itemtype=http://schema.org/PropertyValue itemprop=identifier> <meta itemprop=propertyID content=https://registry.identifiers.org/registry/publisher-id> <span itemprop=name>publisher-id</span><span itemprop=value>30274</span></li> <li itemscope="" itemtype=http://schema.org/PropertyValue itemprop=identifier> <meta itemprop=propertyID content=https://registry.identifiers.org/registry/doi><span itemprop=name>doi</span><span itemprop=value>10.7554/eLife.30274</span></li> <li itemscope="" itemtype=http://schema.org/PropertyValue itemprop=identifier> <meta itemprop=propertyID content=https://registry.identifiers.org/registry/elocation-id> <span itemprop=name>elocation-id</span><span itemprop=value>e30274</span></li> </ul> <section data-itemprop=description> <h2 data-itemtype=http://schema.stenci.la/Heading>Abstract</h2> <meta itemprop=description content="As part of the Reproducibility Project: Cancer Biology, we published a Registered Report (Blum et al., 2015), that described how we intended to replicate selected experiments from the paper ‘Transcriptional amplification in tumor cells with elevated c-Myc’ (Lin et al., 2012). Here we report the results. We found overexpression of c-Myc increased total levels of RNA in P493-6 Burkitt’s lymphoma cells; however, while the effect was in the same direction as the original study (Figure 3E; Lin et al., 2012), statistical significance and the size of the effect varied between the original study and the two different lots of serum tested in this replication. Digital gene expression analysis for a set of genes was also performed on P493-6 cells before and after c-Myc overexpression. Transcripts from genes that were active before c-Myc induction increased in expression following c-Myc overexpression, similar to the original study (Figure 3F; Lin et al., 2012). Transcripts from genes that were silent before c-Myc induction also increased in expression following c-Myc overexpression, while the original study concluded elevated c-Myc had no effect on silent genes (Figure 3F; Lin et al., 2012). Treating the data as paired, we found a statistically significant increase in gene expression for both active and silent genes upon c-Myc induction, with the change in gene expression greater for active genes compared to silent genes. Finally, we report meta-analyses for each result."> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>As part of the <a href=https://osf.io/e81xl/wiki/home/ itemscope="" itemtype=http://schema.stenci.la/Link>Reproducibility Project: Cancer Biology</a>, we published a Registered Report (Blum et al., 2015), that described how we intended to replicate selected experiments from the paper ‘Transcriptional amplification in tumor cells with elevated c-Myc’ (Lin et al., 2012). Here we report the results. We found overexpression of c-Myc increased total levels of RNA in P493-6 Burkitt’s lymphoma cells; however, while the effect was in the same direction as the original study (Figure 3E; Lin et al., 2012), statistical significance and the size of the effect varied between the original study and the two different lots of serum tested in this replication. Digital gene expression analysis for a set of genes was also performed on P493-6 cells before and after c-Myc overexpression. Transcripts from genes that were active before c-Myc induction increased in expression following c-Myc overexpression, similar to the original study (Figure 3F; Lin et al., 2012). Transcripts from genes that were silent before c-Myc induction also increased in expression following c-Myc overexpression, while the original study concluded elevated c-Myc had no effect on silent genes (Figure 3F; Lin et al., 2012). Treating the data as paired, we found a statistically significant increase in gene expression for both active and silent genes upon c-Myc induction, with the change in gene expression greater for active genes compared to silent genes. Finally, we report meta-analyses for each result.</p> </section> <h2 itemscope="" itemtype=http://schema.stenci.la/Heading id=s1>Introduction</h2> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>The <a href=https://osf.io/e81xl/wiki/home/ itemscope="" itemtype=http://schema.stenci.la/Link>Reproducibility Project: Cancer Biology</a> (RP:CB) is a collaboration between the <a href=https://centerforopenscience.org/ itemscope="" itemtype=http://schema.stenci.la/Link>Center for Open Science</a> and <a href=https://www.scienceexchange.com/ itemscope="" itemtype=http://schema.stenci.la/Link>Science Exchange</a> that seeks to address concerns about reproducibility in scientific research by conducting replications of selected experiments from a number of high-profile papers in the field of cancer biology (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib6>Errington et al., 2014</a></cite>). For each of these papers a Registered Report detailing the proposed experimental designs and protocols for the replications was peer reviewed and published prior to data collection. The present paper is a Replication Study that reports the results of the replication experiments detailed in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>) for a 2012 paper by Lin et al., and uses a number of approaches to compare the outcomes of the original experiments and the replications.</p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>In 2012, Lin et al. reported results that the c-Myc transcription factor, a potent oncogene that is frequently overexpressed in a large percentage of cancers, globally amplifies the expression of actively transcribed genes, opposed to regulating specific target genes. Using the P493-6 cell line, a model for <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>MYC</em> activation in Burkitt’s lymphoma, total levels of RNA per cell were reported to increase when c-Myc was highly expressed compared to conditions where c-Myc expression was low. Additionally, active genes in cells with low c-Myc levels were reported to increase in expression upon c-Myc induction, in contrast to genes that were silent under low c-Myc conditions that did not change.</p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>The Registered Report for the 2012 paper by Lin et al. described the experiments to be replicated (Figure 1B and 3E–F), and summarized the current evidence for these findings (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). Since that publication there have been additional studies investigating the ability c-Myc to influence the global gene expression output of cells. Similar to Lin et al. other studies have reported c-Myc dependent amplification of cellular RNA (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib9>Hart et al., 2014</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib12>Hsu et al., 2015</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib19>Nie et al., 2012</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib27>Sabò et al., 2014</a></cite>), although this observation was not reported in all biological systems (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib7>Fagnocchi et al., 2016</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib27>Sabò et al., 2014</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib35>Walz et al., 2014</a></cite>). It has been suggested c-Myc regulates specific genes that indirectly lead to RNA amplification (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib27>Sabò et al., 2014</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib26>Sabò and Amati, 2014</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib35>Walz et al., 2014</a></cite>). This has also been suggested of MYCN (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib5>Duffy et al., 2015</a></cite>). The reported differences could be a result of the intrinsic variation between cell lines in maintaining the transcriptome (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib32>Trakhtenberg et al., 2016</a></cite>). Indeed, a recent study reported that distinct transcriptional regulation can be accounted for by differences in promoter affinity under different c-Myc expression levels (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib18>Lorenzin et al., 2016</a></cite>).</p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>The outcome measures reported in this Replication Study will be aggregated with those from the other Replication Studies to create a dataset that will be examined to provide evidence about reproducibility of cancer biology research, and to identify factors that influence reproducibility more generally.</p> <h2 itemscope="" itemtype=http://schema.stenci.la/Heading id=s2>Results and discussion</h2> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s2-1>Conditional expression of c-Myc in the B-cell line P493-6</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>To test the effects of increased levels of c-Myc on gene expression we used the same human P493-6 B cell line of Burkitt’s lymphoma that contains a conditional tetracycline-repressive <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>MYC</em> transgene (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib23>Pajic et al., 2000</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib30>Schuhmacher et al., 1999</a></cite>) as the original study. We performed Western blot analysis to confirm c-Myc expression could be reduced to very low levels and then reactivated after removal of tetracycline. This is comparable to what was reported in Figure 1B of <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> and described in Protocol 1 in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). Since proliferation of P493-6 cells depend on c-Myc expression and the presence of serum (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib23>Pajic et al., 2000</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib30>Schuhmacher et al., 1999</a></cite>), with serum reported to stimulate a majority of genes independent of c-Myc (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib28>Schlosser et al., 2005</a></cite>), we maintained these cells in separate lots of serum to assess whether the results differed. For cells maintained in both lots of serum, treatment with tetracycline resulted in a strong decrease in c-Myc protein levels (<a href=#fig1 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 1A</a>). After removal of tetracycline, c-Myc levels increased over time approaching the levels observed in tetracycline-free conditions.</p> <figure itemscope="" itemtype=http://schema.stenci.la/Figure id=fig1 title="Figure 1."><label data-itemprop=label>Figure 1.</label><img src=/thema/examples/articleReplication.html.media/fig1.jpg alt="" itemscope="" itemtype=http://schema.org/ImageObject> <figcaption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=induction-of-c-myc-in-p493-6-cells-and-impact-on-total-rna-levels>Induction of c-Myc in P493-6 cells and impact on total RNA levels.</h4> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>P493-6 cells were grown in the presence of tetracycline (Tet) for 72 hr and switched into Tet-free growth medium to induce c-Myc expression. Cells were cultured in two separate lots of serum. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>A</strong>) Representative Western blot using an anti-c-Myc antibody (top panels) or an anti-ß-Actin antibody (bottom panel). Two exposures of the anti-c-Myc antibody are presented to facilitate detection of c-Myc. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>B</strong>) Quantification of total RNA levels (ng of total RNA per 1,000 cells) for cells at 0, 1, and 24 hr after release from Tet. Means reported and error bars represent s.e.m. from three independent biological repeats. For serum lot one, one-way ANOVA on total RNA levels of all groups; <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>F</em>(2, 6)=1.25, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.353. Planned contrast between 0 hr and 24 hr; <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>t</em>(6) = 1.02, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.347 with <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>a priori</em> alpha level = 0.05. For serum lot two, one-way ANOVA on total RNA levels of all groups; <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>F</em>(2, 6)=21.87, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.00176. Planned contrast between 0 hr and 24 hr; <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>t</em>(6) = 5.03, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.0024 with <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>a priori</em> alpha level = 0.05. Additional details for this experiment can be found at <a href=https://osf.io/tfd57/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/tfd57/</a>.</p> </figcaption> </figure> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s2-2>Total RNA levels following c-Myc overexpression</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>We sought to independently replicate whether increased levels of c-Myc resulted in increased absolute levels of RNA. This experiment is similar to what was reported in Figure 3E of <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> and used the same extraction method for total RNA quantification, which was described in Protocol 2 in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). Total RNA was isolated from P493-6 cells 0, 1, and 24 hr after tetracycline release and the amount of RNA per 1,000 cells was quantified (<a href=#fig1 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 1B</a>). We found that under conditions where c-Myc expression was low (0 hr), there was a mean of 1.55 ng total RNA per 1,000 cells (ng/1 k cells) [n = 3, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>SD</em> = 0.20], which increased to 1.77 ng/1 k cells [n = 3, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>SD</em> = 0.31] when c-Myc expression was high (24 hr), a 1.14 times increase, for serum lot one, which was not statistically significant (<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>t</em>(6) = 1.02, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.347). Serum lot two changed from a mean of 1.57 ng/1 k cells [n = 3, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>SD</em> = 0.21] at 0 hr to 2.25 ng/1 k cells [n = 3, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>SD</em> = 0.19] at 24 hr, a 1.43 times increase, which was statistically significant (<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>t</em>(6 = 5.03, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.0024). This compares to the original study, which reported a mean of 4.25 ng/1 k cells at 0 hr, which increased to 5.47 ng/1 k cells at 24 hr, a 1.29 times increase in total RNA levels. In both studies there was a minor decrease at 1 hr after tetracycline release when c-Myc levels begin to become detectable. Total RNA per 1,000 cells at 0 hr were much lower in this replication attempt than those reported in the original study, although changes in total RNA levels were in the same direction following c-Myc expression. Similarly, another independent study that measured total RNA from P493-6 cells reported a different level at 0 hr (~3 ng/1 k cells), while also reporting increased levels following c-Myc expression (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib27>Sabò et al., 2014</a></cite>). There are multiple possible explanations for these differences, such as variation in RNA expression during cell culture passage (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib11>Hiorns et al., 2004</a></cite>), low yield of the RNA isolation procedure (e.g. incomplete homogenization), or the high variance associated with manual cell counts using a hemacytometer (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib2>Biggs and Macmillan, 1948</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib20>Nielson et al., 1991</a></cite>). To summarize, for this experiment we found results that were in the same direction as the original study and not statistically significant for serum lot one, while statistically significant for serum lot two.</p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s2-3>Digital gene expression following c-Myc overexpression</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>To test whether c-Myc expression amplifies the existing gene expression program, digital gene expression analysis using the NanoString nCounter platform was performed on a set of genes from multiple functional categories. This experiment is similar to what was reported in Figure 3F and Table S1 of <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> and described in Protocols 3–4 in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). We quantified mRNA levels/cell of 1369 genes, of which 1212 were the same genes as the 1338 genes interrogated in the original study. We used the same criteria as the original study to classify a gene as silent (expression was less than 0.5 transcript/cell at time 0 hr) or active (more than one transcript/cell at time 0 hr). In cells with low levels of c-Myc (0 hr) there were 708 active genes with a median expression of 4.70, and 580 silent genes with a median expression of 0.032, for serum lot one. For active genes, 75% of the genes increased from 0 hr to 1 hr, 68% increased from 0 hr to 24 hr, and 59% increased from 1 hr to 24 hr upon c-Myc induction. This corresponds to a 1.11, 1.50, and 1.36 times increase in median expression, respectively (<a href=#fig2 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2</a>, <a href=#fig2s1 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2—figure supplement 1</a>). For silent genes, <stencila-code-expression programming-language=r itemscope="" itemtype=http://schema.stenci.la/CodeExpression><code class=r slot=text>round(length(which((silent_1hr_l1-silent_0hr_l1)&gt;0))/length(silent_0hr_l1)*100)</code><output slot=output><span data-itemtype=http://schema.org/Number>74</span></output> </stencila-code-expression>% of the genes increased from 0 hr to 1 hr, <stencila-code-expression programming-language=r itemscope="" itemtype=http://schema.stenci.la/CodeExpression><code class=r slot=text>round(length(which((silent_1hr_l1-silent_0hr_l1)&gt;0))/length(silent_0hr_l1)*100)</code><output slot=output><span data-itemtype=http://schema.org/Number>66</span></output> </stencila-code-expression>% increased from 0 hr to 24 hr, and 50% increased from 1 hr to 24 hr, corresponding to a <stencila-code-expression programming-language=r itemscope="" itemtype=http://schema.stenci.la/CodeExpression><code class=r slot=text>round(median(silent_1hr_l1)/median(silent_0hr_l1),2)</code><output slot=output><span data-itemtype=http://schema.org/Number>1.1</span></output> </stencila-code-expression> and <stencila-code-expression programming-language=r itemscope="" itemtype=http://schema.stenci.la/CodeExpression><code class=r slot=text>round(median(silent_24hr_l1)/median(silent_0hr_l1),2)</code><output slot=output><span data-itemtype=http://schema.org/Number>1.5</span></output> </stencila-code-expression> times increase, and a 0.05 times decrease in median expression, respectively (<a href=#fig2 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2</a>, <a href=#fig2s1 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2—figure supplement 1</a>). Serum lot two gave similar results, although there were variations in the number of genes identified as silent or active as well as the degree of increase among the conditions (<a href=#fig2 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2</a>, <a href=#fig2s1 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2—figure supplement 1</a>). This compares to the original study that identified 755 active genes with a median expression of 7.06, and 514 silent genes with a median expression of 0.00 (more than half the silent genes did not have a reported expression value). Active genes in the original study, increased 91% from 0 hr to 1 hr, 96% from 0 hr to 24 hr, and 87% from 1 hr to 24 hr upon c-Myc induction, corresponding to a 1.23, 2.45, and 1.99 times increase in median expression, respectively. Silent genes in the original study, increased 23% from 0 hr to 1 hr, 29% from 0 hr to 24 hr, and 30% from 1 hr to 24 hr, with the median expression unchanged among conditions. In addition, we further examined the extent of overlap of active and silent genes between the original study and this replication attempt. Of the 1212 genes that were interrogated in both studies, 88.8% (603/679) of the active genes we identified in serum lot one were also active in the original study (90.1% (612/679) for serum lot two). For silent genes, 96.4% (456/473) of the genes we identified as silent in serum lot one were common with the silent genes identified in the original study (95.8% (453/473) for serum lot two).</p> <ol itemscope="" itemtype=http://schema.org/Collection> <li> <figure itemscope="" itemtype=http://schema.stenci.la/Figure id=fig2 title="Figure 2."> <label data-itemprop=label>Figure 2.</label> <stencila-code-chunk itemscope="" itemtype=http://schema.stenci.la/CodeChunk data-programminglanguage=r> <pre class=language-r itemscope="" itemtype=http://schema.stenci.la/CodeBlock slot=text><code># From https://github.com/stencila/examples/tree/master/elife-30274/sources

#Replication Study 48
#Study_48_Protocol_3/4
#R Version 3.4.1

#Required Packages
library(httr) #version 1.2.1
library(rjson) #version 0.2.15
library(ggplot2) #version 2.2.1
library(plyr) #version 1.8.4
library(Rmisc) #version 1.5
library(cowplot) #version 0.7.0
#source(&quot;~/credentials.R&quot;) #for private use during generation

#Downloads R script &quot;download.OSF.file.R&quot;
GET(&quot;https://osf.io/hkpjb/?action=download&quot;, write_disk(&quot;download.OSF.file.R&quot;, overwrite = TRUE))
source(&quot;download.OSF.file.R&quot;)
#calls the download.OSF.file

#Downloads data file &#39;Study_48_Protocols3&amp;4_Combined_Means.csv&#39; from https://osf.io/wmnuf/
download.OSF.file(GUID=&quot;wmnuf&quot;,Access_Token=RPCB_private_access,file_name=&quot;Study_48_Protocols3&amp;4_Combined_Means.csv&quot;)

#reads csv file with all combined means for each lot
comb.means &lt;- read.csv(&quot;Study_48_Protocols3&amp;4_Combined_Means.csv&quot;, header=T, sep=&quot;,&quot;)

comb.means &lt;- comb.means[which(comb.means$Status!=&quot;NA&quot;),] #removes NA status genes
comb.means$lstat &lt;- interaction(comb.means$Time, comb.means$Status) #creates interaction variable between lot and status called &#39;lstat&#39;
comb.means$Time &lt;- as.character(comb.means$Time) #creates a column for Time

active &lt;- comb.means[which(comb.means$Status==&quot;Active&quot;),] #subsets all data on Active gene status
silent &lt;- comb.means[which(comb.means$Status==&quot;Silent&quot;),] #subsets all data on Silent gene status

#create summary data for graph lot 1
active.sum1 &lt;- summarySE(active[which(active$Lot==&quot;C1&quot;),], measurevar=&quot;final.mean&quot;, groupvars=&quot;Time&quot;)
silent.sum1 &lt;- summarySE(silent[which(silent$Lot==&quot;C1&quot;),], measurevar=&quot;final.mean&quot;, groupvars=&quot;Time&quot;)

#create summary data for graph lot 2
active.sum2 &lt;- summarySE(active[which(active$Lot==&quot;C2&quot;),], measurevar=&quot;final.mean&quot;, groupvars=&quot;Time&quot;)
silent.sum2 &lt;- summarySE(silent[which(silent$Lot==&quot;C2&quot;),], measurevar=&quot;final.mean&quot;, groupvars=&quot;Time&quot;)

########## Plots Active Genes/ Lot 1 LOG SCALE ##########
#########################################################

log_activeplot1 &lt;- ggplot(active[which(active$Lot==&quot;C1&quot;),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;Transcripts/Cell&quot;) +
  ggtitle(&quot;Active genes&quot;)+
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(.01,.01),
                     trans = &quot;log2&quot;,
                     limits = c(2^-10,2^11),
                     breaks = c( 2^-9,2^-5,2^-1,2^3,2^7,2^11),
                     labels = c(bquote(&quot;2&quot;^&quot;-9&quot;),bquote(&quot;2&quot;^&quot;-5&quot;),
                                bquote(&quot;2&quot;^&quot;-1&quot;),bquote(&quot;2&quot;^&quot;3&quot;),
                                bquote(&quot;2&quot;^&quot;7&quot;),bquote(&quot;2&quot;^&quot;11&quot;))) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Active Genes/ Lot 1 LINEAR SCALE ##########
############################################################

linear_activeplot1 &lt;- ggplot(active[which(active$Lot==&quot;C1&quot; &amp; active$final.mean&lt;=100),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ggtitle(&quot;Active genes&quot;)+
  ylab(&quot;Transcripts/Cell&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-5, 105),
                     breaks = c(0,20,40,60,80,100)) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Active Genes/ Lot 2 LOG SCALE ##########
#########################################################

log_activeplot2 &lt;- ggplot(active[which(active$Lot==&quot;C2&quot;),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;Transcripts/Cell&quot;) +
  ggtitle(&quot;Active genes&quot;)+
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(.01,.01),
                     trans = &quot;log2&quot;,
                     limits = c(2^-10,2^11),
                     breaks = c( 2^-9,2^-5,2^-1,2^3,2^7,2^11),
                     labels = c(bquote(&quot;2&quot;^&quot;-9&quot;),bquote(&quot;2&quot;^&quot;-5&quot;),
                                bquote(&quot;2&quot;^&quot;-1&quot;),bquote(&quot;2&quot;^&quot;3&quot;),
                                bquote(&quot;2&quot;^&quot;7&quot;),bquote(&quot;2&quot;^&quot;11&quot;))) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Active Genes/ Lot 2 LINEAR SCALE ##########
############################################################

linear_activeplot2 &lt;- ggplot(active[which(active$Lot==&quot;C2&quot; &amp; active$final.mean&lt;=100),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ggtitle(&quot;Active genes&quot;)+
  ylab(&quot;Transcripts/Cell&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-5, 105),
                     breaks = c(0,20,40,60,80,100)) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Silent Genes/ Lot 1 LOG SCALE ##########
#########################################################

log_silentplot1 &lt;- ggplot(silent[which(silent$Lot==&quot;C1&quot;),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;Transcripts/Cell&quot;) +
  ggtitle(&quot;Silent genes&quot;)+
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(.01,.01),
                     trans = &quot;log2&quot;,
                     limits = c(2^-10,2^11),
                     breaks = c( 2^-9,2^-5,2^-1,2^3,2^7,2^11),
                     labels = c(bquote(&quot;2&quot;^&quot;-9&quot;),bquote(&quot;2&quot;^&quot;-5&quot;),
                                bquote(&quot;2&quot;^&quot;-1&quot;),bquote(&quot;2&quot;^&quot;3&quot;),
                                bquote(&quot;2&quot;^&quot;7&quot;),bquote(&quot;2&quot;^&quot;11&quot;))) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Silent Genes/ Lot 1 LINEAR SCALE ##########
############################################################

linear_silentplot1 &lt;- ggplot(silent[which(silent$Lot==&quot;C1&quot; &amp; silent$final.mean&lt;=100),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ggtitle(&quot;Silent genes&quot;)+
  ylab(&quot;Transcripts/Cell&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-5, 105),
                     breaks = c(0,20,40,60,80,100)) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Silent Genes/ Lot 2 LOG SCALE ##########
#########################################################

#plots active cohort 1
log_silentplot2 &lt;- ggplot(silent[which(silent$Lot==&quot;C2&quot;),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;Transcripts/Cell&quot;) +
  ggtitle(&quot;Silent genes&quot;)+
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(.01,.01),
                     trans = &quot;log2&quot;,
                     limits = c(2^-10,2^11),
                     breaks = c( 2^-9,2^-5,2^-1,2^3,2^7,2^11),
                     labels = c(bquote(&quot;2&quot;^&quot;-9&quot;),bquote(&quot;2&quot;^&quot;-5&quot;),
                                bquote(&quot;2&quot;^&quot;-1&quot;),bquote(&quot;2&quot;^&quot;3&quot;),
                                bquote(&quot;2&quot;^&quot;7&quot;),bquote(&quot;2&quot;^&quot;11&quot;))) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))


########## Plots Silent Genes/ Lot 2 LINEAR SCALE ##########
############################################################

linear_silentplot2 &lt;- ggplot(silent[which(silent$Lot==&quot;C2&quot; &amp; silent$final.mean&lt;=100),], aes(x=Time, y = final.mean)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5 ) +
  geom_boxplot(aes(fill=Time), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;)) +
  ggtitle(&quot;Silent genes&quot;)+
  ylab(&quot;Transcripts/Cell&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;0hr&quot;, &quot;1hr&quot;, &quot;24hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-5, 105),
                     breaks = c(0,20,40,60,80,100)) +
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1.88),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5), hjust = 0),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))

###########################################################################################
###########################################################################################

#plots all comparisons for Lot 1 silent
lot1_silent &lt;- subset(comb.means, comb.means$Lot==&quot;C1&quot; &amp; comb.means$Status==&quot;Silent&quot;)
time0 &lt;- subset(lot1_silent, lot1_silent$Time==&quot;0HR&quot;)
time1 &lt;- subset(lot1_silent, lot1_silent$Time==&quot;1HR&quot;)
time24 &lt;- subset(lot1_silent, lot1_silent$Time==&quot;24HR&quot;)
ratio &lt;- c(((log2(time1$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time1$final.mean))))
lot1_silentdat &lt;- as.data.frame(cbind(as.character(lot1_silent[,1]),as.numeric(as.character(ratio))))
lot1_silentdat$V1 &lt;- as.factor(lot1_silentdat$V1)
lot1_silentdat$V3 &lt;- c(rep(&quot;diff1&quot;,nrow(lot1_silent)/3),rep(&quot;diff2&quot;,nrow(lot1_silent)/3),rep(&quot;diff3&quot;,nrow(lot1_silent)/3))
lot1_silentdat$V3 &lt;- as.factor(lot1_silentdat$V3)
lot1_silentdat$V2 &lt;- as.numeric(as.character(lot1_silentdat$V2))
colnames(lot1_silentdat) &lt;- c(&quot;Gene&quot;,&quot;ratio&quot;,&quot;comparison&quot;)

plot_lot1_silent &lt;- ggplot(lot1_silentdat, aes(x=comparison, y = ratio)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5) +
  geom_boxplot(aes(fill=comparison), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;log2 (ratio)&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;1 hr vs. \n 0 hr&quot;, &quot;24 hr vs. \n 0 hr&quot;, &quot;24hr vs. \n 1 hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-4.5,6.5),
                     breaks = c(-4, -2, 0, 2, 4, 6),
                     labels = c(&quot;-4&quot;,&quot;-2&quot;,&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;6&quot;)) +
  geom_hline(yintercept = 0) +
  ggtitle(&quot;Silent genes&quot;)+
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))
plot_lot1_silent

####################################################
####################################################

#plots all comparisons for Lot 2 silent
lot2_silent &lt;- subset(comb.means, comb.means$Lot==&quot;C2&quot; &amp; comb.means$Status==&quot;Silent&quot;)
time0 &lt;- subset(lot2_silent, lot2_silent$Time==&quot;0HR&quot;)
time1 &lt;- subset(lot2_silent, lot2_silent$Time==&quot;1HR&quot;)
time24 &lt;- subset(lot2_silent, lot2_silent$Time==&quot;24HR&quot;)
ratio &lt;- c(((log2(time1$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time1$final.mean))))
lot2_silentdat &lt;- as.data.frame(cbind(as.character(lot2_silent[,1]),as.numeric(as.character(ratio))))
lot2_silentdat$V1 &lt;- as.factor(lot2_silentdat$V1)
lot2_silentdat$V3 &lt;- c(rep(&quot;diff1&quot;,nrow(lot2_silent)/3),rep(&quot;diff2&quot;,nrow(lot2_silent)/3),rep(&quot;diff3&quot;,nrow(lot2_silent)/3))
lot2_silentdat$V3 &lt;- as.factor(lot2_silentdat$V3)
lot2_silentdat$V2 &lt;- as.numeric(as.character(lot2_silentdat$V2))
colnames(lot2_silentdat) &lt;- c(&quot;Gene&quot;,&quot;ratio&quot;,&quot;comparison&quot;)

plot_lot2_silent &lt;- ggplot(lot2_silentdat, aes(x=comparison, y = ratio)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5) +
  geom_boxplot(aes(fill=comparison), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;log2 (ratio)&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;1 hr vs. \n 0 hr&quot;, &quot;24 hr vs. \n 0 hr&quot;, &quot;24hr vs. \n 1 hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-4.5,6.5),
                     breaks = c(-4, -2, 0, 2, 4, 6),
                     labels = c(&quot;-4&quot;,&quot;-2&quot;,&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;6&quot;)) +
  geom_hline(yintercept = 0) +
  ggtitle(&quot;Silent genes&quot;)+
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))
plot_lot2_silent

####################################################
####################################################

#plots all comparisons for Lot 1 active
lot1_active &lt;- subset(comb.means, comb.means$Lot==&quot;C1&quot; &amp; comb.means$Status==&quot;Active&quot;)
time0 &lt;- subset(lot1_active, lot1_active$Time==&quot;0HR&quot;)
time1 &lt;- subset(lot1_active, lot1_active$Time==&quot;1HR&quot;)
time24 &lt;- subset(lot1_active, lot1_active$Time==&quot;24HR&quot;)
ratio &lt;- c(((log2(time1$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time1$final.mean))))
lot1_activedat &lt;- as.data.frame(cbind(as.character(lot1_active[,1]),as.numeric(as.character(ratio))))
lot1_activedat$V1 &lt;- as.factor(lot1_activedat$V1)
lot1_activedat$V3 &lt;- c(rep(&quot;diff1&quot;,nrow(lot1_active)/3),rep(&quot;diff2&quot;,nrow(lot1_active)/3),rep(&quot;diff3&quot;,nrow(lot1_active)/3))
lot1_activedat$V3 &lt;- as.factor(lot1_activedat$V3)
lot1_activedat$V2 &lt;- as.numeric(as.character(lot1_activedat$V2))
colnames(lot1_activedat) &lt;- c(&quot;Gene&quot;,&quot;ratio&quot;,&quot;comparison&quot;)

plot_lot1_active &lt;- ggplot(lot1_activedat, aes(x=comparison, y = ratio)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5) +
  geom_boxplot(aes(fill=comparison), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;log2 (ratio)&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;1 hr vs. \n 0 hr&quot;, &quot;24 hr vs. \n 0 hr&quot;, &quot;24hr vs. \n 1 hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-4.5,6.5),
                     breaks = c(-4, -2, 0, 2, 4, 6),
                     labels = c(&quot;-4&quot;,&quot;-2&quot;,&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;6&quot;)) +
  geom_hline(yintercept = 0) +
  ggtitle(&quot;Active genes&quot;)+
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15,margin = margin(r=10, unit=&quot;pt&quot;)))
plot_lot1_active

####################################################
####################################################

#plots all comparisons for Lot 2 active
lot2_active &lt;- subset(comb.means, comb.means$Lot==&quot;C2&quot; &amp; comb.means$Status==&quot;Active&quot;)
time0 &lt;- subset(lot2_active, lot2_active$Time==&quot;0HR&quot;)
time1 &lt;- subset(lot2_active, lot2_active$Time==&quot;1HR&quot;)
time24 &lt;- subset(lot2_active, lot2_active$Time==&quot;24HR&quot;)
ratio &lt;- c(((log2(time1$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time0$final.mean))),
           ((log2(time24$final.mean))-(log2(time1$final.mean))))
lot2_activedat &lt;- as.data.frame(cbind(as.character(lot2_active[,1]),as.numeric(as.character(ratio))))
lot2_activedat$V1 &lt;- as.factor(lot2_activedat$V1)
lot2_activedat$V3 &lt;- c(rep(&quot;diff1&quot;,nrow(lot2_active)/3),rep(&quot;diff2&quot;,nrow(lot2_active)/3),rep(&quot;diff3&quot;,nrow(lot2_active)/3))
lot2_activedat$V3 &lt;- as.factor(lot2_activedat$V3)
lot2_activedat$V2 &lt;- as.numeric(as.character(lot2_activedat$V2))
colnames(lot2_activedat) &lt;- c(&quot;Gene&quot;,&quot;ratio&quot;,&quot;comparison&quot;)

plot_lot2_active &lt;- ggplot(lot2_activedat, aes(x=comparison, y = ratio)) +
  stat_boxplot(geom =&#39;errorbar&#39;, width=0.5) +
  geom_boxplot(aes(fill=comparison), outlier.shape = 16, outlier.size = 1.5, outlier.colour = &quot;black&quot;, colour = &quot;black&quot;) +
  scale_fill_manual(values=c(&quot;red&quot;, &quot;red&quot;, &quot;red&quot;))+
  ylab(&quot;log2 (ratio)&quot;) +
  xlab(element_blank()) +
  scale_x_discrete(labels=c(&quot;1 hr vs. \n 0 hr&quot;, &quot;24 hr vs. \n 0 hr&quot;, &quot;24hr vs. \n 1 hr&quot;)) +
  scale_y_continuous(expand = c(0,0),
                     limits = c(-4.5,6.5),
                     breaks = c(-4, -2, 0, 2, 4, 6),
                     labels = c(&quot;-4&quot;,&quot;-2&quot;,&quot;0&quot;,&quot;2&quot;,&quot;4&quot;,&quot;6&quot;)) +
  geom_hline(yintercept = 0) +
  ggtitle(&quot;Active genes&quot;)+
  theme_bw()+
  theme(legend.position = &quot;none&quot;,
        axis.ticks.length = unit(0.2, &quot;cm&quot;),
        plot.title = element_text(color = &quot;black&quot;, size = 15, hjust = .5),
        plot.margin = unit(c(1,1,1,1),&quot;cm&quot;),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(),
        panel.background = element_rect(colour = &quot;black&quot;, size=1.8),
        axis.title = element_text(colour = &quot;black&quot;, size = 15),
        axis.text.x = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.text.y = element_text(colour=&quot;black&quot;,size=15, margin=margin(.5,.5,.5,.5)),
        axis.title.x = element_blank(),
        axis.title.y = element_text(colour=&quot;black&quot;,size=15, margin = margin(r=10, unit=&quot;pt&quot;)))
plot_lot2_active

###########################################################################################
###########################################################################################

#combines lot 1 linear scale plots
linear_lot1 &lt;- plot_grid(linear_activeplot1, linear_silentplot1, ncol = 2, align = &quot;h&quot;)
#combines lot 2 linear scale plots
linear_lot2 &lt;- plot_grid(linear_activeplot2, linear_silentplot2, ncol = 2, align = &quot;h&quot;)

#combines lot 1 log scale plots
log_lot1 &lt;- plot_grid(log_activeplot1, log_silentplot1, ncol = 2, align = &quot;h&quot;, labels = c(&quot;A&quot;,&quot;B&quot;))
#combines lot 2 log scale plots
log_lot2 &lt;- plot_grid(log_activeplot2, log_silentplot2, ncol = 2, align = &quot;h&quot;, labels = c(&quot;E&quot;,&quot;F&quot;))

#combines lot 1 ratio plots
ratio_lot1 &lt;- plot_grid(plot_lot1_active, plot_lot1_silent, ncol = 2, align = &quot;h&quot;, labels = c(&quot;C&quot;,&quot;D&quot;))
#combines lot 2 ratio plots
ratio_lot2 &lt;- plot_grid(plot_lot2_active, plot_lot2_silent, ncol = 2, align = &quot;h&quot;, labels = c(&quot;G&quot;,&quot;H&quot;))

#combines Linear Scale Plots
Linear &lt;- plot_grid(linear_lot1,linear_lot2, nrow = 2, align = &quot;h&quot;, labels = c(&quot;Lot 1&quot;,&quot;Lot 2&quot;), label_size = 20)
title &lt;- ggdraw() + draw_label(&quot;Figure 2&quot;, size=20)
figure_2 &lt;- plot_grid(title,Linear,ncol = 1,rel_heights = c(0.1,1))
figure_2
# saves file &#39;Study_48_Figure_2.pdf&#39; locally
ggsave(file = &quot;Study_48_Figure _2.pdf&quot;, width = 8, height = 12)

#combines Ratio Plots
Ratio_Log &lt;- plot_grid(log_lot1,  ratio_lot1, log_lot2, ratio_lot2, ncol = 1, align = &quot;h&quot;)
title &lt;- ggdraw() + draw_label(&quot;Figure 2 - figure supplement 1&quot;, size=20)
figure_2_s1 &lt;- plot_grid(title,Ratio_Log,ncol = 1,rel_heights = c(0.1,1))
figure_2_s1
# saves file &#39;Study_48_Figure_2_figure_supplement_1.pdf&#39; locally
ggsave(file = &quot;Study_48_Figure_2_figure_supplement_1.pdf&quot;, width = 10, height = 18)
</code></pre> <figure slot=outputs><img src=/thema/examples/articleReplication.html.media/fig2.jpg alt="" itemscope="" itemtype=http://schema.org/ImageObject></figure> </stencila-code-chunk> <figcaption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=digital-gene-expression-analysis>Digital gene expression analysis.</h4> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>P493-6 cells grown in the presence of tetracycline (Tet) for 72 hr for repression of the conditional p<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>myc</em>-tet construct, were switched into Tet-free growth medium to induce c-Myc expression. Cells were cultured in two separate lots of serum. Transcripts/cell estimates from NanoString nCounter gene expression assays (1369 genes assay) for active (left) and silent (right) genes at 0, 1, and 24 hr after release from Tet. Active genes expressed greater than one transcript/cell. Silent genes expressed less than 0.5 transcript/cell. Box and whisker plots with median represented as the line through the box and whiskers representing values within 1.5 IQR of the first and third quartile. Cells grown in serum lot one: active genes = 708, silent genes = 580. Cells grown in serum lot two: active genes = 719, silent genes = 573. Confirmatory analysis is reported in <a href=#table1 itemscope="" itemtype=http://schema.stenci.la/Link>Table 1</a> and exploratory statistical analysis is reported in <a href=#table2 itemscope="" itemtype=http://schema.stenci.la/Link>Table 2</a> and <a href=#table3 itemscope="" itemtype=http://schema.stenci.la/Link>Table 3</a>. Additional details for this experiment can be found at <a href=https://osf.io/fn2y4/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/fn2y4/</a>.</p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph></p> <div itemscope="" itemtype=http://schema.org/MediaObject><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <ol data-itemprop=authors></ol> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=list-of-reporter-codesets-and-gene-expression-values>List of Reporter CodeSets and gene expression values.</h4> </div> </figcaption> </figure> </li> <li> <figure itemscope="" itemtype=http://schema.stenci.la/Figure id=fig2s1 title="Figure 2—figure supplement 1."><label data-itemprop=label>Figure 2—figure supplement 1.</label><img src=/thema/examples/articleReplication.html.media/fig2-figsupp1.jpg alt="" itemscope="" itemtype=http://schema.org/ImageObject> <figcaption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=logarithmic-expression-of-genes>Logarithmic expression of genes.</h4> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>This is the same experiment as in <a href=#fig2 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2</a>. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>A–B, E–F</strong>) Gene expression data plotted on a log<sub itemscope="" itemtype=http://schema.stenci.la/Subscript>2</sub> transformed scale for active (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>A, E</strong>) and silent (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>B, F</strong>) genes at 0, 1, and 24 hr after release from Tet for both lots of serum. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>C–D, G–H</strong>) Box and whisker plots showing gene expression changes (log<sub itemscope="" itemtype=http://schema.stenci.la/Subscript>2</sub> ratio) between the indicated times for active (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>C, G</strong>) and silent (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>D, H</strong>) genes. Median represented as the line through the box and whiskers representing values within 1.5 IQR of the first and third quartile. Additional details for this experiment can be found at <a href=https://osf.io/fn2y4/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/fn2y4/</a>.</p> </figcaption> </figure> </li> <li> <figure itemscope="" itemtype=http://schema.stenci.la/Figure id=fig2s2 title="Figure 2—figure supplement 2."><label data-itemprop=label>Figure 2—figure supplement 2.</label><img src=/thema/examples/articleReplication.html.media/fig2-figsupp2.jpg alt="" itemscope="" itemtype=http://schema.org/ImageObject> <figcaption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=comparison-of-gene-expression-data-as-continuous>Comparison of gene expression data as continuous.</h4> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>This is the same experiment as in <a href=#fig2 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2</a>. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>A–C, E–G</strong>) Scatter plots of log<sub itemscope="" itemtype=http://schema.stenci.la/Subscript>2</sub> transformed gene expression data for all genes analyzed at the indicated times on the y and x axes for both lots of serum. Active genes are blue, silent genes are red, and genes that are neither active or silent (expression was more than 0.5 transcript/cell and less than one transcript/cell at time 0 hr) are white. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>D, H</strong>) Box and whisker plots showing gene expression changes (log<sub itemscope="" itemtype=http://schema.stenci.la/Subscript>2</sub> ratio) between the indicated times for all genes analyzed for both lots of serum. Median represented as the line through the box and whiskers representing values within 1.5 IQR of the first and third quartile. Additional details for this experiment can be found at <a href=https://osf.io/fn2y4/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/fn2y4/</a>.</p> </figcaption> </figure> </li> </ol> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>To test whether active genes, as well as silent genes, increased expression during c-Myc induction we performed the confirmatory analysis as outlined in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). This analysis differed from what was reported in the original study by analyzing the data as paired instead of unpaired. As suggested during peer review of the Registered Report, this is because expression of the same gene, analyzed across different conditions, is not independent (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). We performed a Wilcoxon signed-rank test on active genes comparing expression at 0 hr to 1 hr, 0 hr to 24 hr, and 1 hr to 24 hr, which were statistically significant for cells grown in both lots of serum (<a href=#table1 itemscope="" itemtype=http://schema.stenci.la/Link>Table 1</a>). The same comparisons were performed on silent genes, which were also statistically significant, with the exception of the silent gene comparison of 1 hr to 24 hr for serum lot one. Considering this was not the test reported in the original study, we conducted these paired analyses on the original data to provide a direct comparison. For both active and silent genes c-Myc induction resulted in statistically significant increases in expression, with the exception of the silent gene comparison from 0 hr to 1 hr (<a href=#table1 itemscope="" itemtype=http://schema.stenci.la/Link>Table 1</a>). This is in contrast to the results of the unpaired tests that were reported in the original study where active genes were reported to have a statistically significant increase in expression and silent genes were reported as not statistically significant for all comparisons. We conducted an exploratory unpaired analysis on the replication data for comparison, which resulted in statistically significant differences among the active gene comparisons as well as half of the silent gene comparisons (<a href=#table2 itemscope="" itemtype=http://schema.stenci.la/Link>Table 2</a>).</p> <table id=table1 itemscope="" itemtype=http://schema.org/Table> <caption><label data-itemprop=label>Table 1.</label> <div itemprop=caption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=confirmatory-statistical-tests>Confirmatory statistical tests.</h4> </div> </caption> <thead> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Genes</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Comparison</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Study</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Z value</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>P value</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Sample size (n)</th> </tr> </thead> <tbody> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>Active</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 1 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>14.86</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.36e-55</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>708</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>11.83</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>2.83e-34</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>719</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>21.17</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.05e-130</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>755</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>9.922</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>3.67e-24</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>708</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>12.77</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>3.77e-40</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>719</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>23.26</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>3.33e-184</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>755</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>4.742</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.92e-06</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>708</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>10.04</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>9.91e-25</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>719</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>23.26</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>3.33e-184</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>755</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>Silent</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 1 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>12.61</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>7.11e-40</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>580</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>7.05</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>9.29e-13</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>572</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>−1.998</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.0457</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>274</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>8.328</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>2.22e-17</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>579</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>8.156</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.03e-16</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>572</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>3.179</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.00144</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>276</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.6853</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.493</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>580</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>4.436</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>8.35e-06</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>573</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>5.806</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>4.11e-09</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>236</td> </tr> </tbody> </table> <table id=table2 itemscope="" itemtype=http://schema.org/Table> <caption><label data-itemprop=label>Table 2.</label> <div itemprop=caption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=exploratory-statistical-tests>Exploratory statistical tests.</h4> </div> </caption> <thead> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Genes</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Comparison</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Study</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>W value</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>P value</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Sample size (n)</th> </tr> </thead> <tbody> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>Active</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 1 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>270378</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.0103</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1416</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>274696</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.0395</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1438</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>318799</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>6.67e-05</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1510</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>300774</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>7.16e-11</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1416</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>324564</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>4.74e-17</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1438</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>400999</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.16e-42</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1510</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>281679</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>5.45e-05</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1416</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>308954</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.45e-10</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1438</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>372714</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>4.11e-25</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1510</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>Silent</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 1 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>187682</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.000638</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1160</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>174695</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.0602</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1146</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>127104</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.236</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1028</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>185804</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.00203</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1160</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>184470</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.000289</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1146</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>132082</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.997</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1028</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>166122</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.716</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1160</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>173608</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.0918</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1146</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>136443</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0.295</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1028</td> </tr> </tbody> </table> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>Importantly, though, the question of whether the change in expression among active genes is different than silent genes has not been tested. This would require a separate test on their difference (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib8>Gelman and Stern, 2006</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib21>Nieuwenhuis et al., 2011</a></cite>). To test whether active genes increased in expression during c-Myc induction more than silent genes, we performed an exploratory analysis on the difference in expression of active genes during c-Myc induction (e.g. from 0 hr to 24 hr) compared to the difference in expression of silent genes over that same period (e.g. from 0 hr to 24 hr). For both the original and replication data, there was a statistically significant increase in expression of active genes compared to silent genes (<a href=#table3 itemscope="" itemtype=http://schema.stenci.la/Link>Table 3</a>). This suggests that active genes and silent genes do not have similar rates of expression upon c-Myc induction. To summarize, for this experiment we found results that were in the same direction as the original study and suggest that while both active and silent genes increased in expression upon c-Myc induction, the rate of increase was different.</p> <table id=table3 itemscope="" itemtype=http://schema.org/Table> <caption><label data-itemprop=label>Table 3.</label> <div itemprop=caption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=exploratory-statistical-tests-1>Exploratory statistical tests.</h4> </div> </caption> <thead> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Comparison</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Study</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>W value</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>P value</th> <th itemscope="" itemtype=http://schema.stenci.la/TableCell>Sample size (n)</th> </tr> </thead> <tbody> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 1 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>303897</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>7.78e-50</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1288</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>278646</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.1e-27</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1292</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>349351</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.27e-130</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1269</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>0 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>272441</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>5.18e-24</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1288</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>292865</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>7.4e-39</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1292</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>368182</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1.14e-163</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1269</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1 hr vs 24 hr</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 1</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>235077</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>7.45e-06</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1288</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>RP:CB Lot 2</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>272028</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>3.73e-23</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1292</td> </tr> <tr itemscope="" itemtype=http://schema.stenci.la/TableRow> <td itemscope="" itemtype=http://schema.stenci.la/TableCell></td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell><cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite> </td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>332069</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>5.72e-104</td> <td itemscope="" itemtype=http://schema.stenci.la/TableCell>1269</td> </tr> </tbody> </table> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>The original study and this replication attempt used the same criteria to characterize a gene as silent or active, but there are many negative consequences of dichotomizing continuous variables, such as information loss, especially with a small gene set (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib1>Altman and Royston, 2006</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib4>Cohen, 1983</a></cite>). Papers published after the original study took an unbiased view by collecting comprehensive RNA-sequencing data to assess if the transcriptional effects of c-Myc were direct or indirect, concluding c-Myc activates and represses transcription of discrete gene sets, which in turn leads to induced RNA amplification (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib27>Sabò et al., 2014</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib35>Walz et al., 2014</a></cite>). Furthermore, Sabò and colleagues also used NanoString technology to quantify a subset of the differentially expressed genes identified by RNA-seq and observed similar results that revealed upward shifts in gene expression upon c-Myc induction (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib27>Sabò et al., 2014</a></cite>). However, instead of dichotomizing genes as active or silent, gene expression data was presented as continuous. Similarly, we presented the digital gene expression data generated during this replication attempt as continuous, which illustrates a general pattern of overall increased gene expression following c-Myc induction (<a href=#fig2s2 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2—figure supplement 2</a>). Importantly, though, these results are limited to the 1369 genes interrogated in this study and may or may not reflect how the entire transcriptome of P493-6 cells respond to c-Myc induction. </p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s2-4>Meta-analyses of original and replicated effects</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>We performed a meta-analysis using a random-effects model to combine each of the effects described above as pre-specified in the confirmatory analysis plan (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). To provide a standardized measure of the effect, a common effect size was calculated for each effect from the original and replication studies. Cohen’s <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>d</em> is the standardized difference between two means using the pooled sample standard deviation. The effect size <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>r</em> is a standardized measure of the strength and direction of the association between two variables, in this case time during c-Myc induction and gene expression. The estimate of the effect size of one study, as well as the associated uncertainty (i.e. confidence interval), compared to the effect size of the other study provides another approach to compare the original and replication results (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib6>Errington et al., 2014</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib33>Valentine et al., 2011</a></cite>). Importantly, the width of the confidence interval for each study is a reflection of not only the confidence level (e.g. 95%), but also variability of the sample (e.g. <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>SD</em>) and sample size.</p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>The comparison of total RNA levels at low levels of c-Myc (0 hr) compared to high levels of c-Myc (24 hr) resulted in <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>d</em> = 4.19, 95% CI [0.94, 7.37] for the data reported in Figure 3E of the original study (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al., 2012</a></cite>). This compares to <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>d</em> = 0.83, 95% CI [−0.91, 2.48] for serum lot one and <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>d</em> = 4.11, 95% CI [0.90, 7.23] for serum lot two reported in this study. A meta-analysis (<a href=#fig3 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 3A</a>) of these effects resulted in <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>d</em> = 2.52, 95% CI [0.01, 5.03], which was statistically significant (<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.0488). The original and replication results are consistent when considering the direction of the effect, which suggests c-Myc induction increases total RNA levels in P493-6 Burkitt’s lymphoma cells. Noticeably, there was substantial within-study variation observed in this replication attempt, due the different serum lots tested. The point estimate of serum lot one was not within the confidence intervals of the original study and serum lot two, and vice versa; however the point estimate of the original study and serum lot two were within the confidence intervals of each other.</p> <figure itemscope="" itemtype=http://schema.stenci.la/Figure id=fig3 title="Figure 3."><label data-itemprop=label>Figure 3.</label> <stencila-code-chunk itemscope="" itemtype=http://schema.stenci.la/CodeChunk data-programminglanguage=r> <pre class=language-r itemscope="" itemtype=http://schema.stenci.la/CodeBlock slot=text><code># From https://github.com/stencila/examples/tree/master/elife-30274/sources

# Replication Study 48
# Meta-analysis Forest plots
# Figure 3 Forestplots
# R version 3.3.2

# Required Packages
library(httr) #version 1.2.1
library(rjson) #version 0.2.15
library(ggplot2) #version 2.2.1
library(cowplot) #version 0.7.0
#source(&quot;~/credentials.R&quot;) #for private use during generation

#Downloads R script &quot;download.OSF.file.R&quot;
GET(&quot;https://osf.io/hkpjb/?action=download&quot;, write_disk(&quot;download.OSF.file.R&quot;, overwrite = TRUE))
source(&quot;download.OSF.file.R&quot;)
#calls the download.OSF.file

#Downloads data file &#39;Study_48_Meta_Analysis.csv&#39; from https://osf.io/zmhqy/
download.OSF.file(GUID=&quot;zmhqy&quot;,Access_Token=RPCB_private_access,file_name=&quot;Study_48_Meta_Analysis.csv&quot;)

#reads csv file
meta &lt;- read.csv(&quot;Study_48_Meta_Analysis.csv&quot;, header = T)

#######################################
#re-orders the data frame
meta &lt;- meta[order(meta$study),]
meta &lt;- meta[order(meta$comparison),]

#subsets data to plot results
d1 &lt;- subset(meta[1:4,]) #Active 0v1
d2 &lt;- subset(meta[5:8,]) #Active 0v24
d3 &lt;- subset(meta[9:12,]) #Active 1v24

d4 &lt;- subset(meta[13:16,]) #Silent 0v1
d5 &lt;- subset(meta[17:20,]) #Silent 0v24
d6 &lt;- subset(meta[21:24,]) #Silent 1v24

d7 &lt;- subset(meta[25:28,]) #protocol 2 oh vs. 24hr

#Plots for protocol 3 analyses:
########################### Active 0hr vs. 1hr #####################################

#re-order the levels for plotting
desired_order &lt;- c(&quot;Meta-Analysis&quot;,&quot;RP:CB Lot 2&quot;,&quot;RP:CB Lot 1&quot;, &quot;Lin et al., 2012&quot; )

#re-orders data for plotting
d1$study &lt;- factor(as.character(d1$study), levels=desired_order)
d1 &lt;- d1[order(d1$study),]

a1 &lt;- ggplot(data=d1,aes(x=estimate,y=d1$study)) +
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-.2,1)) +
  scale_x_continuous(breaks= c(-.2,0,.2,.4,.6,.8,1)) +
  ylab(d1$comparison[1])+
  xlab(NULL)+
  scale_y_discrete(labels = c(paste(as.character(d1$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d1$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d1$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d1$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d1$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d1$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d1$N[4]),&quot;)&quot;))))) +
  theme(panel.background = element_blank(),
        axis.ticks.x=element_blank(),
        axis.title.x = element_blank(),
        axis.text.x = element_blank(),
        axis.text.y = element_text(size=15),
        legend.position=&quot;none&quot;,
        axis.line.x = element_blank(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.line.y = element_line(),
        plot.margin = unit(c(.5 ,2,.075,.5), &quot;in&quot;))
a1 &lt;- ggdraw(a1)+
  draw_text(paste(&quot;r&quot;,&quot;[&quot;,&quot;L.CI&quot;, &quot;,&quot;, &quot;U.CI&quot;, &quot;]&quot;), x = .84, y = 0.91, fontface=&quot;bold&quot;)+
  draw_text(paste(formatC(d1$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d1$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d1$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.785, size = 12, hjust=0)+
  draw_text(paste(formatC(d1$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d1$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d1$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.575, size = 12, hjust=0)+
  draw_text(paste(formatC(d1$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d1$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d1$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y =  0.3575, size = 12, hjust=0)+
  draw_text(paste(formatC(d1$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d1$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d1$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.154, size = 12, hjust=0)

########################## Active 0hr vs. 24hr #####################################

#re-order the levels for plotting
d2$study &lt;- factor(as.character(d2$study), levels=desired_order)
d2 &lt;- d2[order(d2$study),]

a2 &lt;- ggplot(data=d2,aes(x=estimate,y=d2$study)) +
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-.2,1)) +
  scale_x_continuous(breaks= c(-.2,0,.2,.4,.6,.8,1)) +
  ylab(d2$comparison[1])+
  xlab(NULL)+
  scale_y_discrete(labels = c(paste(as.character(d2$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d2$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d2$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d2$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d2$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d2$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d2$N[4]),&quot;)&quot;)))))+
  theme(panel.background = element_blank(),
        axis.ticks.x=element_blank(),
        axis.title.x = element_blank(),
        axis.text.x = element_blank(),
        axis.text.y = element_text(size=15),
        legend.position=&quot;none&quot;,
        axis.line.x = element_blank(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.line.y = element_line(),
        plot.margin = unit(c(.22,2,.22,.5), &quot;in&quot;))
a2 &lt;- ggdraw(a2)+
  draw_text(paste(formatC(d2$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d2$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d2$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.829, size = 12, hjust=0)+
  draw_text(paste(formatC(d2$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d2$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d2$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.615, size = 12, hjust=0)+
  draw_text(paste(formatC(d2$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d2$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d2$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y =  0.395, size = 12, hjust=0)+
  draw_text(paste(formatC(d2$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d2$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d2$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.183, size = 12, hjust=0)

########################### Active 1hr vs. 24hr #####################################

#re-order the levels for plotting
d3$study &lt;- factor(as.character(d3$study), levels=desired_order)
d3 &lt;- d3[order(d3$study),]

a3 &lt;- ggplot(data=d3,aes(x=estimate,y=d3$study))+
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-.2,1)) +
  scale_x_continuous(breaks= c(-.2,0,.2,.4,.6,.8,1)) +
  ylab(d3$comparison[1]) +
  xlab(&quot;r&quot;) +
  scale_y_discrete(labels = c(paste(as.character(d3$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d3$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d3$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d3$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d3$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d3$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d3$N[4]),&quot;)&quot;))))) +
  theme(panel.background = element_blank(),
        legend.position=&quot;none&quot;,
        axis.line.x = element_line(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.text.y = element_text(size=15),
        axis.title.x = element_text(size=15),
        axis.line.y = element_line(),
        plot.margin = unit(c(.1, 2,0,.5), &quot;in&quot;))
a3 &lt;- ggdraw(a3)+
  draw_text(paste(formatC(d3$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d3$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d3$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.858, size = 12, hjust=0)+
  draw_text(paste(formatC(d3$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d3$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d3$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.6445, size = 12, hjust=0)+
  draw_text(paste(formatC(d3$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d3$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d3$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y =  0.432, size = 12, hjust=0)+
  draw_text(paste(formatC(d3$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d3$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d3$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.218, size = 12, hjust=0)

#Plots for protocol 3 analyses Silent genes:
########################### Silent 0hr vs. 1hr #####################################

#re-order the levels for plotting
d4$study &lt;- factor(as.character(d4$study), levels=desired_order)
d4 &lt;- d4[order(d4$study),]

s1 &lt;- ggplot(data=d4,aes(x=estimate,y=d4$study)) +
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-.2,1)) +
  scale_x_continuous(breaks= c(-.2,0,.2,.4,.6,.8,1)) +
  ylab(d4$comparison[1])+
  xlab(NULL)+
  scale_y_discrete(labels = c(paste(as.character(d4$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d4$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d4$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d4$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d4$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d4$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d4$N[4]),&quot;)&quot;)))))+
  theme(panel.background = element_blank(),
        axis.ticks.x=element_blank(),
        axis.title.x = element_blank(),
        axis.text.x = element_blank(),
        axis.text.y = element_text(size=15),
        legend.position=&quot;none&quot;,
        axis.line.x = element_blank(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.line.y = element_line(),
        plot.margin = unit(c(.5 ,2,.075,.5), &quot;in&quot;))
s1 &lt;- ggdraw(s1)+
  draw_text(paste(&quot;r&quot;,&quot;[&quot;,&quot;L.CI&quot;, &quot;,&quot;, &quot;U.CI&quot;, &quot;]&quot;), x = .84, y = 0.91, fontface=&quot;bold&quot;)+
  draw_text(paste(formatC(d4$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d4$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d4$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.785, size = 12, hjust=0)+
  draw_text(paste(formatC(d4$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d4$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d4$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.575, size = 12, hjust=0)+
  draw_text(paste(formatC(d4$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d4$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d4$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y =  0.3575, size = 12, hjust=0)+
  draw_text(paste(formatC(d4$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d4$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d4$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.154, size = 12, hjust=0)


########################## Silent 0hr vs. 24hr #####################################

#re-order the levels for plotting
d5$study &lt;- factor(as.character(d5$study), levels=desired_order)
d5 &lt;- d5[order(d5$study),]

s2 &lt;- ggplot(data=d5,aes(x=estimate,y=d5$study)) +
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-.2,1)) +
  scale_x_continuous(breaks= c(-.2,0,.2,.4,.6,.8,1)) +
  ylab(d5$comparison[1])+
  xlab(NULL)+
  scale_y_discrete(labels = c(paste(as.character(d5$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d5$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d5$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d5$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d5$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d5$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d5$N[4]),&quot;)&quot;)))))+
  theme(panel.background = element_blank(),
        axis.ticks.x=element_blank(),
        axis.title.x = element_blank(),
        axis.text.x = element_blank(),
        axis.text.y = element_text(size=15),
        legend.position=&quot;none&quot;,
        axis.line.x = element_blank(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.line.y = element_line(),
        plot.margin = unit(c(.22,2,.22,.5), &quot;in&quot;))
s2 &lt;- ggdraw(s2)+
  draw_text(paste(formatC(d5$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d5$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d5$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.829, size = 12, hjust=0)+
  draw_text(paste(formatC(d5$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d5$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d5$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.615, size = 12, hjust=0)+
  draw_text(paste(formatC(d5$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d5$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d5$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y =  0.395, size = 12, hjust=0)+
  draw_text(paste(formatC(d5$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d5$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d5$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.183, size = 12, hjust=0)

########################## Silent 1hr vs. 24hr #####################################

#re-order the levels for plotting
d6$study &lt;- factor(as.character(d6$study), levels=desired_order)
d6 &lt;- d6[order(d6$study),]

s3 &lt;- ggplot(data=d6,aes(x=estimate,y=factor(d6$study)))+
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-.2,1)) +
  scale_x_continuous(breaks= c(-.2,0,.2,.4,.6,.8,1)) +
  ylab(d6$comparison[1]) +
  xlab(&quot;r&quot;) +
  scale_y_discrete(labels = c(paste(as.character(d6$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d6$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d6$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d6$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d6$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d6$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d6$N[4]),&quot;)&quot;))))) +
  theme(panel.background = element_blank(),
        legend.position=&quot;none&quot;,
        axis.line.x = element_line(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.text.y = element_text(size=15),
        axis.title.x = element_text(size=15),
        axis.line.y = element_line(),
        plot.margin = unit(c(.1, 2,0,.5), &quot;in&quot;))
s3 &lt;- ggdraw(s3)+
  draw_text(paste(formatC(d6$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d6$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d6$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.858, size = 12, hjust=0)+
  draw_text(paste(formatC(d6$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d6$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d6$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.6445, size = 12, hjust=0)+
  draw_text(paste(formatC(d6$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d6$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d6$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y =  0.432, size = 12, hjust=0)+
  draw_text(paste(formatC(d6$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d6$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d6$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.75, y = 0.218, size = 12, hjust=0)

########################## Protocol 2 Meta Analysis #####################################

#re-order the levels for plotting
d7$study &lt;- factor(as.character(d7$study), levels=desired_order)
d7 &lt;- d7[order(d7$study),]

pro2 &lt;- ggplot(data=d7,aes(x=estimate,y=d7$study))+
  geom_point(size=5, colour=&quot;black&quot;, fill = &quot;black&quot;, shape = c(22,21,21,23)) +
  geom_errorbarh(aes(xmin=CI.lower,xmax=CI.upper, height = .1)) +
  geom_vline(xintercept=0,linetype=&quot;dashed&quot;) +
  coord_cartesian(xlim=c(-1,8)) +
  scale_x_continuous(breaks= c(-1,0,1,2,3,4,5,6,7,8)) +
  ylab(d7$comparison[1]) +
  xlab(&quot;Cohen&#39;s&quot;~italic(&quot;d&quot;)) +
  scale_y_discrete(labels = c(paste(as.character(d7$study[1])),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d7$study[2]),
                                                   paste(&quot;x (n =&quot;, as.character(d7$N[2]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d7$study[3]),
                                                   paste(&quot;x (n =&quot;, as.character(d7$N[3]),&quot;)&quot;))),
                              gsub(&quot;x&quot;, &quot;\n&quot;,paste(as.character(d7$study[4]),
                                                   paste(&quot;x (n =&quot;, as.character(d7$N[4]),&quot;)&quot;))))) +
  theme(panel.background = element_blank(),
        legend.position=&quot;none&quot;,
        axis.line.x = element_line(),
        axis.title.y = element_text(size=15,margin=margin(0,30,0,0)),
        axis.text.y = element_text(size=15),
        axis.line.y = element_line(),
        plot.margin = margin(.6, 8, .5, .55, &quot;in&quot;))
pro2 &lt;- ggdraw(pro2)+
  draw_text(paste(&quot;Cohen&#39;s d&quot;,&quot;[&quot;,&quot;L.CI&quot;, &quot;,&quot;, &quot;U.CI&quot;, &quot;]&quot;), x = 0.568, y = 0.93, fontface=&quot;bold&quot;) +
  draw_text(paste(formatC(d7$estimate[[4]], digits =  2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d7$CI.lower[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d7$CI.upper[[4]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.52, y = 0.79, size = 12, hjust=0)+
  draw_text(paste(formatC(d7$estimate[[3]], digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d7$CI.lower[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d7$CI.upper[[3]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.52, y = 0.62, size = 12, hjust=0)+
  draw_text(paste(formatC(d7$estimate[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d7$CI.lower[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d7$CI.upper[[2]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.52, y =  0.455, size = 12, hjust=0)+
  draw_text(paste(formatC(d7$estimate[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;[&quot;,
                  formatC(d7$CI.lower[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;,&quot;,
                  formatC(d7$CI.upper[[1]],digits = 2, format = &quot;f&quot;),
                  &quot;]&quot;), x = 0.52, y = 0.285, size = 12, hjust=0)


#Creates figure for protocol 3 meta-analyses
pro3 &lt;- plot_grid(a1, s1, a2, s2, a3, s3, nrow = 3)

figure &lt;- plot_grid(pro2, pro3, nrow = 2, rel_heights = c(1,3), labels = c(&quot;A&quot;, &quot;B&quot;), label_size = 25)
title &lt;- ggdraw() + draw_label(&quot;Figure 3&quot;, size=20)
figure_3 &lt;- plot_grid(title,figure,ncol = 1,rel_heights = c(0.1,1))
figure_3

################################################################

# saves file &#39;Study_48_Figure_3_forestplots.pdf&#39; locally
ggsave(file = &quot;Study_48_Figure_3_forestplots.pdf&quot;, width = 16, height = 24)
</code></pre> <figure slot=outputs><img src=/thema/examples/articleReplication.html.media/fig3.jpg alt="" itemscope="" itemtype=http://schema.org/ImageObject></figure> </stencila-code-chunk> <figcaption> <h4 itemscope="" itemtype=http://schema.stenci.la/Heading id=meta-analyses-of-each-effect> Meta-analyses of each effect.</h4> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>Effect size and 95% confidence interval are presented for <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al., 2012</a></cite>, this replication study (RP:CB), and a random effects meta-analysis of those two effects. Cohen’s <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>d</em> is the standardized difference between the two measurements, with a larger positive value indicating total RNA levels are increased at 24 hr compared to 0 hr. The effect size <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>r</em> is a standardized measure of the correlation (strength and direction) of the association between gene expression and c-Myc induction, with a larger positive value indicating gene expression increased during the course of c-Myc induction. Sample sizes used in <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al., 2012</a></cite> and this replication attempt are reported under the study name. (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>A</strong>) Total RNA levels in P493-6 cells 0 hr compared to 24 hr after release from tetracycline (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 0.0488). (<strong itemscope="" itemtype=http://schema.stenci.la/Strong>B</strong>) Gene expression of active or silent genes are shown for all comparisons. Active genes: 0 hr compared to 1 hr (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 1.12x10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>-7</sup>), 0 hr compared to 24 hr (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 7.01x10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>-4</sup>), 1 hr compared to 24 hr (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 0.0129). Silent genes: 0 hr compared to 1 hr (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 0.203), 0 hr compared to 24 hr (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 7.10x10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>-17</sup>), 1 hr compared to 24 hr (meta-analysis <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em> = 0.0571). Additional details for these meta-analyses can be found at <a href=https://osf.io/5yscz/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/5yscz/</a>.</p> </figcaption> </figure> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>There were six comparisons of the gene expression data, three for active genes and three for silent genes (<a href=#fig3 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 3B</a>). These calculations were performed analyzing the data as paired, for reasons discussed above and as prespecified in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). For active genes, expression at 0 hr to 1 hr, 0 hr to 24 hr, and 1 hr to 24 hr the meta-analyses were statistically significant (<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=1.12×10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>−7</sup>, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=7.01×10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>−4</sup>, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.0129, respectively). In all comparisons the results were consistent when considering the direction of the effect; however the effect size point estimate of each study (original, replication serum lot one, replication serum lot two) was not within the confidence interval of the other studies. Further, the large confidence intervals of the meta-analysis along with statistically significant Cochran’s <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>Q</em> tests suggest heterogeneity between the original and replication studies. For silent genes, the meta-analysis was not statistically significant for gene expression at 0 hr to 1 hr and 1 hr to 24 hr (<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.203, <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=0.0571, respectively) and the effect size point estimate of each study was not within the confidence interval of the other studies. Similar to the active gene comparisons, the large confidence intervals of the meta-analysis along with statistically significant Cochran’s <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>Q</em> tests suggest heterogeneity between the studies. Furthermore, for the 0 hr to 1 hr comparison the original study and replication studies were in opposite directions, while the 1 hr to 24 hr comparison was consistent. Finally, the comparison between 0 hr and 24 hr for silent genes was consistent when considering direction of the effect with a statistically significant meta-analysis (<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>p</em>=7.10×10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>−17</sup>). The point estimate of the original study was not within the confidence intervals of the replication studies; however both replication studies with different serum lots were within the confidence intervals of the original study and each other. Overall, the gene expression analysis indicates that the effect sizes observed from the two serum lots tested in this replication attempt, although not identical, were more similar to each other than to the original study.</p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>This direct replication provides an opportunity to understand the present evidence of these effects. Any known differences, including reagents and protocol differences, were identified prior to conducting the experimental work and described in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). However, this is limited to what was obtainable from the original paper and through communication with the original authors, which means there might be particular features of the original experimental protocol that could be critical, but unidentified. So while some aspects, such as the cell line, induction time course, and the method used to measure gene expression were maintained, others were changed at the time of study design (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>) that could affect results, such as the analytical approach (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib31>Silberzahn et al., 2017</a></cite>) and serum lot (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib15>Leek et al., 2010</a></cite>). Furthermore, other aspects were unknown or not easily controlled for. These include variables such as cell line genetic drift (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib13>Hughes et al., 2007</a></cite>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib14>Kleensang et al., 2016</a></cite>) or changes in cellular volume that can impact overall transcript abundance (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib22>Padovan-Merhar et al., 2015</a></cite>). Whether these or other factors influence the outcomes of this study is open to hypothesizing and further investigation, which is facilitated by direct replications and transparent reporting.</p> <h2 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3>Materials and methods</h2> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>As described in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>), we attempted a replication of the experiments reported in Figures 1B and 3E-F of <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib17>Lin et al. (2012)</a></cite>. A detailed description of all protocols can be found in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). Additional detailed experimental notes, data, and analysis are available on the Open Science Framework (OSF) (RRID:<a href=https://scicrunch.org/resolver/SCR_003238 itemscope="" itemtype=http://schema.stenci.la/Link>SCR_003238</a>) (<a href=https://osf.io/mokeb/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/mokeb/</a>; <cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib16>Lewis et al., 2017</a></cite>). This includes the R Markdown file (<a href=https://osf.io/vdrsh/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/vdrsh/</a>) that was used to compose this manuscript, which is a reproducible document linking the results in the article directly to the data and code that produced them (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib10>Hartgerink, 2017</a></cite>).</p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3-1>Cell culture</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>P493-6 cells (shared by Young lab, Whitehead Institute for Biomedical Research, RRID: <a href=https://scicrunch.org/resolver/CVCL_6783 itemscope="" itemtype=http://schema.stenci.la/Link>CVCL_6783</a>) were maintained in RPMI-1640 supplemented with 1% Ala-Gln and 10% tetracycline-free FBS (Clontech, Mountain View, CA, cat# 631105, lot# 1: A15003, lot# 2: A15032). Cells were grown at 37°C in a humidified atmosphere at 5% CO<sub itemscope="" itemtype=http://schema.stenci.la/Subscript>2</sub>. Quality control data for the cell line are available at <a href=https://osf.io/e6ftz/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/e6ftz/.</a> This includes results confirming the cell line was free of mycoplasma contamination (DDC Medical, Fairfield, Ohio). Additionally, STR DNA profiling of the cell line was performed (DDC Medical, Fairfield, Ohio). </p> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>For repression of the conditional p<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>myc</em>-tet construct in P493-6 cells, 0.1 µg/ml tetracycline (Sigma-Aldrich, St. Louis, MO, cat# T7660) was added to the culture medium and cells were incubated for 72 hr. Under these conditions, P493-6 cells did not proliferate due to a dependency on the expression of <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>MYC</em> (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib30>Schuhmacher et al., 1999</a></cite>). For <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>MYC</em> re-induction, cells were washed three times with growth medium and grown in tetracycline-free culture conditions. </p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3-2>Western blot</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>P493-6 cells were harvested at the indicated times and total cell lysates were prepared by pelleting ~1×10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>7</sup> cells (determined with a C-chip disposable hemocytometer) at 4°C at 1,200 rpm for 5 min using a refrigerated centrifuge (Eppendorf, Westbury, NY, model# 5810R). After cell pellets were washed once with ice-cold 1X PBS, pellets were resuspended in RIPA lysis buffer containing 2X SIGMAFAST Protease inhibitors and 2X Phosphatase inhibitor cocktails 2 and 3. Protein concentrations were determined using the Bradford assay according to the manufacturer’s instructions. Sample buffer was added to protein lysates and 50 µg of protein along with protein ladder was resolved by SDS-PAGE and transferred to PVDF membrane as described in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). The membrane was blocked with 5% w/v nonfat dry milk in 1X TBS with 0.2% Tween-20 (TBST). Membranes were probed with rabbit anti-c-Myc [clone Y69] (Epitomics, Burlingame, CA, cat# 1472–1; RRID:<a href=https://scicrunch.org/resolver/AB_731658 itemscope="" itemtype=http://schema.stenci.la/Link>AB_731658</a>); 1:5000 dilution in 5% w/v nonfat dry milk/TBST and mouse anti-ß-actin [clone AC-15] (Sigma-Aldrich, cat# A5441; RRID:<a href=https://scicrunch.org/resolver/AB_476744 itemscope="" itemtype=http://schema.stenci.la/Link>AB_476744</a>); 1:10,000 dilution in 5% w/v nonfat dry milk/TBST. Each incubation was followed by washes with TBST and the appropriate secondary antibody: HRP-conjugated donkey anti-rabbit (Sigma-Aldrich, cat# GERPN2124); 1:10,000 dilution in 5% w/v nonfat dry milk/TBST or HRP-conjugated sheep anti-mouse (Sigma-Aldrich, cat# GERPN2124); 1:10,000 dilution in 5% w/v nonfat dry milk/TBST. Membranes were washed with TBST and incubated with ECL Prime Chemiluminescent reagent (Sigma-Aldrich, cat# GERPN2232) according to the manufacturer’s instructions. Western blot images were acquired with G:BOX iChem XT and GeneSnap software (RRID:<a href=https://scicrunch.org/resolver/SCR_014249 itemscope="" itemtype=http://schema.stenci.la/Link>SCR_014249</a>), version 7.12.02 (Syngene, Frederick, Maryland) and quantified using ImageJ software (RRID:<a href=https://scicrunch.org/resolver/SCR_003070 itemscope="" itemtype=http://schema.stenci.la/Link>SCR_003070</a>), version 1.50i (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib29>Schneider et al., 2012</a></cite>). All images taken are available at <a href=https://osf.io/ujg7t/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/ujg7t/.</a></p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3-3>RNA quantification</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>P493-6 cells were harvested at the indicated times and total RNA extraction was performed by pelleting ~1×10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>7</sup> cells (exact number determined with a C-chip disposable hemocytometer) and homogenizing the sample in 1 ml Tri Reagent (Sigma-Aldrich, cat# T9424) according to the manufacturer’s instructions. For each sample 10% v/v miRNA Homogenate Additive was added, vortexed, and incubated on ice for 10 min. For each 1 ml of Tri Reagent, 100 µl of bromochloropropane was added, vortexed for 15–30 s, incubated for 5 min at RT, then centrifuged at 12,000x<em itemscope="" itemtype=http://schema.stenci.la/Emphasis>g</em> for 10 min at 4°C. The aqueous phase was recovered and total RNA isolation was performed using the miRVana miRNA extraction kit (Ambion, Waltham, MA, cat# AM1561) according to the manufacturer’s instructions. Recovered RNA was eluated in 100 µl nuclease-free water. Total RNA concentrations and purity (data available at <a href=https://osf.io/jh5r4/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/jh5r4/</a>) were measured using a NanoDrop ND-1000 (Thermo Fisher Scientific, Waltham, Massachusetts) with the NanoDrop Operating Software, version 3.3, and converted to ng per 1,000 cells.</p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3-4>RNA extraction and NanoString nCounter digital gene expression assay</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>P493-6 cells were harvested at the indicated times and 1 × 10<sup itemscope="" itemtype=http://schema.stenci.la/Superscript>6</sup> cells were collected (number determined with a C-chip disposable hemocytometer) and lysed directly in 100 µl Buffer RLT (Qiagen, Hilden, Germany, cat# 79216) supplemented with ß-mercaptoethanol to yield a concentration of 10,000 cells per µl. This was performed four independent times. Multiple 4 µl aliquots were stored and shipped at −80°C with temperature monitored during shipping to avoid freeze/thaw cycles. Lysates were processed according to the Cell Lysate Protocol (nCounter Gene Expression Assay Manual, NanoString Technologies, Seattle, Washington) according to the manufacturer’s instructions and as described in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). Three nCounter Reporter CodeSets (nCounter GX Human Immunology Kit, nCounter GX Human Kinase Kit, nCounter Custom CodeSet) encompassing 1369 genes across multiple functional categories were used. Following hybridization, samples were immediately processed with the nCounter Analysis System (NanoString Technologies, NCT-PREP-120). The count data was collected using the nCounter RCC Collector Worksheet (NanoString Technologies), version 1.6.0 and then positive-, negative-, and housekeeping gene-normalized per nCounter guidelines. Expression for each gene was averaged across the four independent replicate samples. Additionally, for genes that appeared on multiple CodeSets, expression values were averaged together to generate a single value for each gene. A gene was defined as transcriptionally active if its average expression was above one transcript/cell at 0 hr and transcriptionally silent if below 0.5 transcript/cell. A list of all Reporter CodeSets and their expression values (transcripts/cell) are available at <a href=#fig2sdata1 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 2—source data 1</a>. Additional files and analysis scripts are available at <a href=https://osf.io/fn2y4/. itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/fn2y4/.</a></p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3-5>Statistical analysis</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>Statistical analysis was performed with R software (RRID:<a href=https://scicrunch.org/resolver/SCR_001905 itemscope="" itemtype=http://schema.stenci.la/Link>SCR_001905</a>), version 3.3.2 (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib25>R Core Team, 2017</a></cite>). All data, csv files, and analysis scripts are available on the OSF (<a href=https://osf.io/mokeb/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/mokeb/</a>). Confirmatory statistical analysis was pre-registered (<a href=https://osf.io/nj8wb/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/nj8wb/</a>) before the experimental work began as outlined in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>). Proposed analysis of gene expression data was conducted by the Wilcoxon signed-rank test using the method proposed by Pratt to handle zero differences (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib24>Pratt, 1959</a></cite>), with additional exploratory analysis performed using the Wilcoxon rank sum test as reported in the original study and a Wilcoxon rank sum test on the difference in expression of active genes during c-Myc induction (e.g. from 0 hr to 24 hr) compared to the difference in expression of silent genes over that same period (e.g. from 0 hr to 24 hr). Data were checked to ensure assumptions of statistical tests were met. When described in the results, the Bonferroni correction, to account for multiple testings, was applied to the alpha error by dividing the uncorrected value (.05) by the number of tests performed. Although the Bonferroni method is conservative, it was accounted for in the power calculations to ensure sample size was sufficient. In cases where the number of groups were three and the sample sizes were evenly distributed among the groups, Fisher&#39;s LSD test was performed resulting in an <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>a priori</em> significance threshold of. 05. A meta-analysis of a common original and replication effect size was performed with a random effects model and the <em itemscope="" itemtype=http://schema.stenci.la/Emphasis>metafor</em> package (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib34>Viechtbauer, 2010</a></cite>) (available at: <a href=https://osf.io/5yscz/ itemscope="" itemtype=http://schema.stenci.la/Link>https://osf.io/5yscz/</a>). The sample sizes reported in <a href=#table1 itemscope="" itemtype=http://schema.stenci.la/Link>Table 1</a> and <a href=#fig3 itemscope="" itemtype=http://schema.stenci.la/Link>Figure 3</a> for the gene analysis comparisons is based on the sample size used in the Wilcoxon signed-rank test, which removes samples with zero differences after ranking (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib24>Pratt, 1959</a></cite>). The raw original study data were shared by the original authors with the summary data published in the Registered Report (<cite itemscope="" itemtype=http://schema.stenci.la/Cite><a href=#bib3>Blum et al., 2015</a></cite>) and was used in the power calculations to determine the sample size for this study.</p> <h3 itemscope="" itemtype=http://schema.stenci.la/Heading id=s3-6>Deviations from registered report</h3> <p itemscope="" itemtype=http://schema.stenci.la/Paragraph>The number of flasks, and thus cells, was increased when tetracycline was added to P493-6 cells to account for the cells not proliferating during this period (i.e. there were two Flask B’s as described in the Registered Report, which were pooled prior to seeding). The proposed statistical analysis for the western blot analysis (Protocol 1) described in the Registered Report was not performed since the levels of normalized c-Myc at time 0 hr was at the limit of detection. The number of genes analyzed in the original study, and thus listed in the Registered Report, was reported incorrectly as 1388 instead of 1338 (data shared by original authors). NanoString analysis was conducted using the nCounter RCC Collector Worksheet instead of nSolver Analysis software. Additionally, the statistical tests reported in Figure 3F of the original study incorrectly described the comparisons as between 0 hr and 1 hr instead of between 0 hr and 24 hr (scripts shared by original authors). The corrected values are described above for comparisons and used in the meta-analysis. Additional materials and instrumentation not listed in the Registered Report, but needed during experimentation are also listed.</p> <section data-itemprop=references> <h2 data-itemtype=http://schema.stenci.la/Heading>References</h2> <ol> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib1> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DG Altman"><span data-itemprop=givenNames><span itemprop=givenName>DG</span></span><span data-itemprop=familyNames><span itemprop=familyName>Altman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="P Royston"><span data-itemprop=givenNames><span itemprop=givenName>P</span></span><span data-itemprop=familyNames><span itemprop=familyName>Royston</span></span></li> </ol><time itemprop=datePublished datetime=2006>2006</time><span itemprop=headline>The cost of dichotomising continuous variables</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>332</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>BMJ</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=The%20cost%20of%20dichotomising%20continuous%20variables"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib2> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Biggs"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Biggs</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RL Macmillan"><span data-itemprop=givenNames><span itemprop=givenName>RL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Macmillan</span></span></li> </ol><time itemprop=datePublished datetime=1948>1948</time><span itemprop=headline>The error of the red cell count</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>1</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Journal of Clinical Pathology</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>288</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>291</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=The%20error%20of%20the%20red%20cell%20count"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib3> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Blum"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Blum</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Hao"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hao</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M McCarthy"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>McCarthy</span></span></li> </ol><time itemprop=datePublished datetime=2015>2015</time><span itemprop=headline>Registered report: Transcriptional amplification in tumor cells with elevated c-Myc</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>4</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>eLife</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Registered%20report:%20Transcriptional%20amplification%20in%20tumor%20cells%20with%20elevated%20c-Myc"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib4> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Cohen"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Cohen</span></span></li> </ol><time itemprop=datePublished datetime=1983>1983</time><span itemprop=headline>The cost of dichotomization</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>7</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Applied Psychological Measurement</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>249</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>253</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=The%20cost%20of%20dichotomization"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib5> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DJ Duffy"><span data-itemprop=givenNames><span itemprop=givenName>DJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Duffy</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Krstic"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Krstic</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Halasz"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Halasz</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Schwarzl"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schwarzl</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Fey"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fey</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Iljin"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Iljin</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JP Mehta"><span data-itemprop=givenNames><span itemprop=givenName>JP</span></span><span data-itemprop=familyNames><span itemprop=familyName>Mehta</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Killick"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Killick</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Whilde"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Whilde</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Turriziani"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Turriziani</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Haapa-Paananen"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Haapa-Paananen</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="V Fey"><span data-itemprop=givenNames><span itemprop=givenName>V</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fey</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Fischer"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fischer</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Westermann"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Westermann</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K-O Henrich"><span data-itemprop=givenNames><span itemprop=givenName>K-O</span></span><span data-itemprop=familyNames><span itemprop=familyName>Henrich</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Bannert"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bannert</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DG Higgins"><span data-itemprop=givenNames><span itemprop=givenName>DG</span></span><span data-itemprop=familyNames><span itemprop=familyName>Higgins</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="W Kolch"><span data-itemprop=givenNames><span itemprop=givenName>W</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kolch</span></span></li> </ol><time itemprop=datePublished datetime=2015>2015</time><span itemprop=headline content="Integrative omics reveals MYCN as a global suppressor of cellular signalling and enables network-based therap…">Integrative omics reveals MYCN as a global suppressor of cellular signalling and enables network-based therapeutic target discovery in neuroblastoma</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>6</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Oncotarget</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>43182</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>43201</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Integrative%20omics%20reveals%20MYCN%20as%20a%20global%20suppressor%20of%20cellular%20signalling%20and%20enables%20network-based%20therap%E2%80%A6"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib6> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TM Errington"><span data-itemprop=givenNames><span itemprop=givenName>TM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Errington</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Iorns"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Iorns</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="W Gunn"><span data-itemprop=givenNames><span itemprop=givenName>W</span></span><span data-itemprop=familyNames><span itemprop=familyName>Gunn</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="FE Tan"><span data-itemprop=givenNames><span itemprop=givenName>FE</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tan</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Lomax"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lomax</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="BA Nosek"><span data-itemprop=givenNames><span itemprop=givenName>BA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nosek</span></span></li> </ol><time itemprop=datePublished datetime=2014>2014</time><span itemprop=headline>An open investigation of the reproducibility of cancer biology research</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>3</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>eLife</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=An%20open%20investigation%20of%20the%20reproducibility%20of%20cancer%20biology%20research"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib7> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Fagnocchi"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fagnocchi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Cherubini"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Cherubini</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Hatsuda"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hatsuda</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Fasciani"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fasciani</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Mazzoleni"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Mazzoleni</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="V Poli"><span data-itemprop=givenNames><span itemprop=givenName>V</span></span><span data-itemprop=familyNames><span itemprop=familyName>Poli</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="V Berno"><span data-itemprop=givenNames><span itemprop=givenName>V</span></span><span data-itemprop=familyNames><span itemprop=familyName>Berno</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RL Rossi"><span data-itemprop=givenNames><span itemprop=givenName>RL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rossi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Reinbold"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Reinbold</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Endele"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Endele</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Schroeder"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schroeder</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Rocchigiani"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rocchigiani</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Ż Szkarłat"><span data-itemprop=givenNames><span itemprop=givenName>Ż</span></span><span data-itemprop=familyNames><span itemprop=familyName>Szkarłat</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Oliviero"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Oliviero</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Dalton"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Dalton</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Zippo"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Zippo</span></span></li> </ol><time itemprop=datePublished datetime=2016>2016</time><span itemprop=headline>A Myc-driven self-reinforcing regulatory network maintains mouse embryonic stem cell identity</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>7</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature Communications</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=A%20Myc-driven%20self-reinforcing%20regulatory%20network%20maintains%20mouse%20embryonic%20stem%20cell%20identity"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib8> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Gelman"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Gelman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Stern"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Stern</span></span></li> </ol><time itemprop=datePublished datetime=2006>2006</time><span itemprop=headline>The difference between “significant” and “not significant” is not itself statistically significant</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>60</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>The American Statistician</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>328</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>331</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=The%20difference%20between%20%E2%80%9Csignificant%E2%80%9D%20and%20%E2%80%9Cnot%20significant%E2%80%9D%20is%20not%20itself%20statistically%20significant"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib9> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JR Hart"><span data-itemprop=givenNames><span itemprop=givenName>JR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hart</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TC Roberts"><span data-itemprop=givenNames><span itemprop=givenName>TC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Roberts</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MS Weinberg"><span data-itemprop=givenNames><span itemprop=givenName>MS</span></span><span data-itemprop=familyNames><span itemprop=familyName>Weinberg</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KV Morris"><span data-itemprop=givenNames><span itemprop=givenName>KV</span></span><span data-itemprop=familyNames><span itemprop=familyName>Morris</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="PK Vogt"><span data-itemprop=givenNames><span itemprop=givenName>PK</span></span><span data-itemprop=familyNames><span itemprop=familyName>Vogt</span></span></li> </ol><time itemprop=datePublished datetime=2014>2014</time><span itemprop=headline>MYC regulates the non-coding transcriptome</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>5</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Oncotarget</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>12543</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>12554</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=MYC%20regulates%20the%20non-coding%20transcriptome"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib10> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CHJ Hartgerink"><span data-itemprop=givenNames><span itemprop=givenName>CHJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hartgerink</span></span></li> </ol><time itemprop=datePublished datetime=2017>2017</time><span itemprop=headline>Composing reproducible manuscripts using R Markdown</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>eLife</span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Composing%20reproducible%20manuscripts%20using%20R%20Markdown"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib11> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LR Hiorns"><span data-itemprop=givenNames><span itemprop=givenName>LR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hiorns</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TD Bradshaw"><span data-itemprop=givenNames><span itemprop=givenName>TD</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bradshaw</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LA Skelton"><span data-itemprop=givenNames><span itemprop=givenName>LA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Skelton</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Q Yu"><span data-itemprop=givenNames><span itemprop=givenName>Q</span></span><span data-itemprop=familyNames><span itemprop=familyName>Yu</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LR Kelland"><span data-itemprop=givenNames><span itemprop=givenName>LR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kelland</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Leyland-Jones"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Leyland-Jones</span></span></li> </ol><time itemprop=datePublished datetime=2004>2004</time><span itemprop=headline>Variation in RNA expression and genomic DNA content acquired during cell culture</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>90</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>British Journal of Cancer</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>476</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>482</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Variation%20in%20RNA%20expression%20and%20genomic%20DNA%20content%20acquired%20during%20cell%20culture"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib12> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TY Hsu"><span data-itemprop=givenNames><span itemprop=givenName>TY</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hsu</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LM Simon"><span data-itemprop=givenNames><span itemprop=givenName>LM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Simon</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="NJ Neill"><span data-itemprop=givenNames><span itemprop=givenName>NJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Neill</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Marcotte"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Marcotte</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Sayad"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sayad</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CS Bland"><span data-itemprop=givenNames><span itemprop=givenName>CS</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bland</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="GV Echeverria"><span data-itemprop=givenNames><span itemprop=givenName>GV</span></span><span data-itemprop=familyNames><span itemprop=familyName>Echeverria</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Sun"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sun</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SJ Kurley"><span data-itemprop=givenNames><span itemprop=givenName>SJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kurley</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Tyagi"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tyagi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KL Karlin"><span data-itemprop=givenNames><span itemprop=givenName>KL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Karlin</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Dominguez-Vidaña"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Dominguez-Vidaña</span></span> </li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JD Hartman"><span data-itemprop=givenNames><span itemprop=givenName>JD</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hartman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Renwick"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Renwick</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Scorsone"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Scorsone</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RJ Bernardi"><span data-itemprop=givenNames><span itemprop=givenName>RJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bernardi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SO Skinner"><span data-itemprop=givenNames><span itemprop=givenName>SO</span></span><span data-itemprop=familyNames><span itemprop=familyName>Skinner</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Jain"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Jain</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Orellana"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Orellana</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Lagisetti"><span data-itemprop=givenNames><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lagisetti</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="I Golding"><span data-itemprop=givenNames><span itemprop=givenName>I</span></span><span data-itemprop=familyNames><span itemprop=familyName>Golding</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SY Jung"><span data-itemprop=givenNames><span itemprop=givenName>SY</span></span><span data-itemprop=familyNames><span itemprop=familyName>Jung</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JR Neilson"><span data-itemprop=givenNames><span itemprop=givenName>JR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Neilson</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="XH Zhang"><span data-itemprop=givenNames><span itemprop=givenName>XH</span></span><span data-itemprop=familyNames><span itemprop=familyName>Zhang</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TA Cooper"><span data-itemprop=givenNames><span itemprop=givenName>TA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Cooper</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TR Webb"><span data-itemprop=givenNames><span itemprop=givenName>TR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Webb</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="BG Neel"><span data-itemprop=givenNames><span itemprop=givenName>BG</span></span><span data-itemprop=familyNames><span itemprop=familyName>Neel</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CA Shaw"><span data-itemprop=givenNames><span itemprop=givenName>CA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Shaw</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TF Westbrook"><span data-itemprop=givenNames><span itemprop=givenName>TF</span></span><span data-itemprop=familyNames><span itemprop=familyName>Westbrook</span></span></li> </ol><time itemprop=datePublished datetime=2015>2015</time><span itemprop=headline>The spliceosome is a therapeutic vulnerability in MYC-driven cancer</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>525</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>384</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>388</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=The%20spliceosome%20is%20a%20therapeutic%20vulnerability%20in%20MYC-driven%20cancer"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib13> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="P Hughes"><span data-itemprop=givenNames><span itemprop=givenName>P</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hughes</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Marshall"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Marshall</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Y Reid"><span data-itemprop=givenNames><span itemprop=givenName>Y</span></span><span data-itemprop=familyNames><span itemprop=familyName>Reid</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Parkes"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Parkes</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Gelber"><span data-itemprop=givenNames><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Gelber</span></span></li> </ol><time itemprop=datePublished datetime=2007>2007</time><span itemprop=headline>The costs of using unauthenticated</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>43</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>over-passaged cell lines: how much more data do we need? BioTechniques</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>575</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>586</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=The%20costs%20of%20using%20unauthenticated"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib14> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Kleensang"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kleensang</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MM Vantangoli"><span data-itemprop=givenNames><span itemprop=givenName>MM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Vantangoli</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Odwin-DaCosta"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Odwin-DaCosta</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="ME Andersen"><span data-itemprop=givenNames><span itemprop=givenName>ME</span></span><span data-itemprop=familyNames><span itemprop=familyName>Andersen</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Boekelheide"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Boekelheide</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Bouhifd"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bouhifd</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="AJ Fornace"><span data-itemprop=givenNames><span itemprop=givenName>AJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fornace</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="HH Li"><span data-itemprop=givenNames><span itemprop=givenName>HH</span></span><span data-itemprop=familyNames><span itemprop=familyName>Li</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CB Livi"><span data-itemprop=givenNames><span itemprop=givenName>CB</span></span><span data-itemprop=familyNames><span itemprop=familyName>Livi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Madnick"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Madnick</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Maertens"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Maertens</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Rosenberg"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rosenberg</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JD Yager"><span data-itemprop=givenNames><span itemprop=givenName>JD</span></span><span data-itemprop=familyNames><span itemprop=familyName>Yager</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Zhaog"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Zhaog</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Hartung"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hartung</span></span></li> </ol><time itemprop=datePublished datetime=2016>2016</time><span itemprop=headline content="Genetic variability in a frozen batch of MCF-7 cells invisible in routine authentication affecting cell funct…">Genetic variability in a frozen batch of MCF-7 cells invisible in routine authentication affecting cell function</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>6</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Scientific Reports</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Genetic%20variability%20in%20a%20frozen%20batch%20of%20MCF-7%20cells%20invisible%20in%20routine%20authentication%20affecting%20cell%20funct%E2%80%A6"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib15> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JT Leek"><span data-itemprop=givenNames><span itemprop=givenName>JT</span></span><span data-itemprop=familyNames><span itemprop=familyName>Leek</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RB Scharpf"><span data-itemprop=givenNames><span itemprop=givenName>RB</span></span><span data-itemprop=familyNames><span itemprop=familyName>Scharpf</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="HC Bravo"><span data-itemprop=givenNames><span itemprop=givenName>HC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bravo</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Simcha"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Simcha</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Langmead"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Langmead</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="WE Johnson"><span data-itemprop=givenNames><span itemprop=givenName>WE</span></span><span data-itemprop=familyNames><span itemprop=familyName>Johnson</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Geman"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Geman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Baggerly"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Baggerly</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RA Irizarry"><span data-itemprop=givenNames><span itemprop=givenName>RA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Irizarry</span></span></li> </ol><time itemprop=datePublished datetime=2010>2010</time><span itemprop=headline>Tackling the widespread and critical impact of batch effects in high-throughput data</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>11</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature Reviews Genetics</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>733</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>739</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Tackling%20the%20widespread%20and%20critical%20impact%20of%20batch%20effects%20in%20high-throughput%20data"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib16> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LM Lewis"><span data-itemprop=givenNames><span itemprop=givenName>LM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lewis</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MC Edwards"><span data-itemprop=givenNames><span itemprop=givenName>MC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Edwards</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="ZR Meyers"><span data-itemprop=givenNames><span itemprop=givenName>ZR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Meyers</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CC Talbot"><span data-itemprop=givenNames><span itemprop=givenName>CC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Talbot</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Hao"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hao</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Blum"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Blum</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Iorns"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Iorns</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Tsui"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tsui</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Denis"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Denis</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="N Perfito"><span data-itemprop=givenNames><span itemprop=givenName>N</span></span><span data-itemprop=familyNames><span itemprop=familyName>Perfito</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TM Errington"><span data-itemprop=givenNames><span itemprop=givenName>TM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Errington</span></span></li> </ol><time itemprop=datePublished datetime=2017>2017</time><span itemprop=headline>Study 48: Replication of Lin et al., 2012 (Cell)</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Open Science Framework</span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Study%2048:%20Replication%20of%20Lin%20et%20al.,%202012%20(Cell)"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib17> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CY Lin"><span data-itemprop=givenNames><span itemprop=givenName>CY</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lin</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Lovén"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lovén</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="PB Rahl"><span data-itemprop=givenNames><span itemprop=givenName>PB</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rahl</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RM Paranal"><span data-itemprop=givenNames><span itemprop=givenName>RM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Paranal</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CB Burge"><span data-itemprop=givenNames><span itemprop=givenName>CB</span></span><span data-itemprop=familyNames><span itemprop=familyName>Burge</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JE Bradner"><span data-itemprop=givenNames><span itemprop=givenName>JE</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bradner</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TI Lee"><span data-itemprop=givenNames><span itemprop=givenName>TI</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lee</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RA Young"><span data-itemprop=givenNames><span itemprop=givenName>RA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Young</span></span></li> </ol><time itemprop=datePublished datetime=2012>2012</time><span itemprop=headline>Transcriptional amplification in tumor cells with elevated c-Myc</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>151</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Cell</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>56</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>67</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Transcriptional%20amplification%20in%20tumor%20cells%20with%20elevated%20c-Myc"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib18> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Lorenzin"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lorenzin</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="U Benary"><span data-itemprop=givenNames><span itemprop=givenName>U</span></span><span data-itemprop=familyNames><span itemprop=familyName>Benary</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Baluapuri"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Baluapuri</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Walz"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Walz</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LA Jung"><span data-itemprop=givenNames><span itemprop=givenName>LA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Jung</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B von Eyss"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>von</span><span itemprop=familyName>Eyss</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Kisker"><span data-itemprop=givenNames><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kisker</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Wolf"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wolf</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Eilers"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eilers</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Wolf"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wolf</span></span></li> </ol><time itemprop=datePublished datetime=2016>2016</time><span itemprop=headline>Different promoter affinities account for specificity in MYC-dependent gene regulation</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>5</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>eLife</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Different%20promoter%20affinities%20account%20for%20specificity%20in%20MYC-dependent%20gene%20regulation"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib19> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Z Nie"><span data-itemprop=givenNames><span itemprop=givenName>Z</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nie</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="G Hu"><span data-itemprop=givenNames><span itemprop=givenName>G</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hu</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="G Wei"><span data-itemprop=givenNames><span itemprop=givenName>G</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wei</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Cui"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Cui</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Yamane"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Yamane</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="W Resch"><span data-itemprop=givenNames><span itemprop=givenName>W</span></span><span data-itemprop=familyNames><span itemprop=familyName>Resch</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Wang"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wang</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DR Green"><span data-itemprop=givenNames><span itemprop=givenName>DR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Green</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Tessarollo"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tessarollo</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Casellas"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Casellas</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Zhao"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Zhao</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Levens"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Levens</span></span></li> </ol><time itemprop=datePublished datetime=2012>2012</time><span itemprop=headline>c-Myc is a universal amplifier of expressed genes in lymphocytes and embryonic stem cells</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>151</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Cell</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>68</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>79</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=c-Myc%20is%20a%20universal%20amplifier%20of%20expressed%20genes%20in%20lymphocytes%20and%20embryonic%20stem%20cells"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib20> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Nielson"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nielson</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="G Smyth"><span data-itemprop=givenNames><span itemprop=givenName>G</span></span><span data-itemprop=familyNames><span itemprop=familyName>Smyth</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="P Greenfield"><span data-itemprop=givenNames><span itemprop=givenName>P</span></span><span data-itemprop=familyNames><span itemprop=familyName>Greenfield</span></span></li> </ol><time itemprop=datePublished datetime=1991>1991</time><span itemprop=headline>Hemacytometer Cell Count Distributions: Implications of Non-Poisson Behavior</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>7</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Biotechnology Progress</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>560</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>563</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Hemacytometer%20Cell%20Count%20Distributions:%20Implications%20of%20Non-Poisson%20Behavior"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib21> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Nieuwenhuis"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nieuwenhuis</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="BU Forstmann"><span data-itemprop=givenNames><span itemprop=givenName>BU</span></span><span data-itemprop=familyNames><span itemprop=familyName>Forstmann</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="EJ Wagenmakers"><span data-itemprop=givenNames><span itemprop=givenName>EJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wagenmakers</span></span></li> </ol><time itemprop=datePublished datetime=2011>2011</time><span itemprop=headline>Erroneous analyses of interactions in neuroscience: a problem of significance</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>14</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature Neuroscience</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>1105</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>1107</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Erroneous%20analyses%20of%20interactions%20in%20neuroscience:%20a%20problem%20of%20significance"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib22> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="O Padovan-Merhar"><span data-itemprop=givenNames><span itemprop=givenName>O</span></span><span data-itemprop=familyNames><span itemprop=familyName>Padovan-Merhar</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="GP Nair"><span data-itemprop=givenNames><span itemprop=givenName>GP</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nair</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="AG Biaesch"><span data-itemprop=givenNames><span itemprop=givenName>AG</span></span><span data-itemprop=familyNames><span itemprop=familyName>Biaesch</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Mayer"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Mayer</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Scarfone"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Scarfone</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SW Foley"><span data-itemprop=givenNames><span itemprop=givenName>SW</span></span><span data-itemprop=familyNames><span itemprop=familyName>Foley</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="AR Wu"><span data-itemprop=givenNames><span itemprop=givenName>AR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wu</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LS Churchman"><span data-itemprop=givenNames><span itemprop=givenName>LS</span></span><span data-itemprop=familyNames><span itemprop=familyName>Churchman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Singh"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Singh</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Raj"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Raj</span></span></li> </ol><time itemprop=datePublished datetime=2015>2015</time><span itemprop=headline content="Single mammalian cells compensate for differences in cellular volume and DNA copy number through independent …">Single mammalian cells compensate for differences in cellular volume and DNA copy number through independent global transcriptional mechanisms</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>58</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Molecular Cell</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>339</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>352</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Single%20mammalian%20cells%20compensate%20for%20differences%20in%20cellular%20volume%20and%20DNA%20copy%20number%20through%20independent%20%E2%80%A6"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib23> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Pajic"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pajic</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Spitkovsky"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Spitkovsky</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Christoph"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Christoph</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Kempkes"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kempkes</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Schuhmacher"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schuhmacher</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MS Staege"><span data-itemprop=givenNames><span itemprop=givenName>MS</span></span><span data-itemprop=familyNames><span itemprop=familyName>Staege</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Brielmeier"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Brielmeier</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Ellwart"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Ellwart</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Kohlhuber"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kohlhuber</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="GW Bornkamm"><span data-itemprop=givenNames><span itemprop=givenName>GW</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bornkamm</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Polack"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Polack</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Eick"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eick</span></span></li> </ol><time itemprop=datePublished datetime=2000>2000</time><span itemprop=headline>Cell cycle activation by c-myc in a burkitt lymphoma model cell line</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>87</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>International Journal of Cancer</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>787</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>793</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Cell%20cycle%20activation%20by%20c-myc%20in%20a%20burkitt%20lymphoma%20model%20cell%20line"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib24> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JW Pratt"><span data-itemprop=givenNames><span itemprop=givenName>JW</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pratt</span></span></li> </ol><time itemprop=datePublished datetime=1959>1959</time><span itemprop=headline>Remarks on zeros and ties in the wilcoxon signed rank procedures</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>54</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Journal of the American Statistical Association</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>655</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>667</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Remarks%20on%20zeros%20and%20ties%20in%20the%20wilcoxon%20signed%20rank%20procedures"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib25> <ol data-itemprop=authors></ol><time itemprop=datePublished datetime=2017>2017</time><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>R Foundation for Statistical Computing</span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text="> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib26> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Sabò"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sabò</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Amati"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Amati</span></span></li> </ol><time itemprop=datePublished datetime=2014>2014</time><span itemprop=headline>Genome recognition by MYC</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>4</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Cold Spring Harbor Perspectives in Medicine</span></span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Genome%20recognition%20by%20MYC"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib27> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Sabò"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sabò</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TR Kress"><span data-itemprop=givenNames><span itemprop=givenName>TR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kress</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Pelizzola"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pelizzola</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S de Pretis"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>de</span><span itemprop=familyName>Pretis</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MM Gorski"><span data-itemprop=givenNames><span itemprop=givenName>MM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Gorski</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Tesi"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tesi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MJ Morelli"><span data-itemprop=givenNames><span itemprop=givenName>MJ</span></span><span data-itemprop=familyNames><span itemprop=familyName>Morelli</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="P Bora"><span data-itemprop=givenNames><span itemprop=givenName>P</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bora</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Doni"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Doni</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Verrecchia"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Verrecchia</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Tonelli"><span data-itemprop=givenNames><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Tonelli</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="G Fagà"><span data-itemprop=givenNames><span itemprop=givenName>G</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fagà</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="V Bianchi"><span data-itemprop=givenNames><span itemprop=givenName>V</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bianchi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Ronchi"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Ronchi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Low"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Low</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Müller"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Müller</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Guccione"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Guccione</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Campaner"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Campaner</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Amati"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Amati</span></span></li> </ol><time itemprop=datePublished datetime=2014>2014</time><span itemprop=headline>Selective transcriptional regulation by Myc in cellular growth control and lymphomagenesis</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>511</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>488</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>492</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Selective%20transcriptional%20regulation%20by%20Myc%20in%20cellular%20growth%20control%20and%20lymphomagenesis"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib28> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="I Schlosser"><span data-itemprop=givenNames><span itemprop=givenName>I</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schlosser</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Hölzel"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hölzel</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Hoffmann"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hoffmann</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Burtscher"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Burtscher</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Kohlhuber"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kohlhuber</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Schuhmacher"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schuhmacher</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Chapman"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Chapman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="UH Weidle"><span data-itemprop=givenNames><span itemprop=givenName>UH</span></span><span data-itemprop=familyNames><span itemprop=familyName>Weidle</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Eick"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eick</span></span></li> </ol><time itemprop=datePublished datetime=2005>2005</time><span itemprop=headline>Dissection of transcriptional programmes in response to serum and c-Myc in a human B-cell line</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>24</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Oncogene</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>520</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>524</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Dissection%20of%20transcriptional%20programmes%20in%20response%20to%20serum%20and%20c-Myc%20in%20a%20human%20B-cell%20line"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib29> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CA Schneider"><span data-itemprop=givenNames><span itemprop=givenName>CA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schneider</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="WS Rasband"><span data-itemprop=givenNames><span itemprop=givenName>WS</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rasband</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KW Eliceiri"><span data-itemprop=givenNames><span itemprop=givenName>KW</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eliceiri</span></span></li> </ol><time itemprop=datePublished datetime=2012>2012</time><span itemprop=headline>NIH Image to ImageJ: 25 years of image analysis</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>9</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature Methods</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>671</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>675</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib30> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Schuhmacher"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schuhmacher</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MS Staege"><span data-itemprop=givenNames><span itemprop=givenName>MS</span></span><span data-itemprop=familyNames><span itemprop=familyName>Staege</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Pajic"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pajic</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Polack"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Polack</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="UH Weidle"><span data-itemprop=givenNames><span itemprop=givenName>UH</span></span><span data-itemprop=familyNames><span itemprop=familyName>Weidle</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="GW Bornkamm"><span data-itemprop=givenNames><span itemprop=givenName>GW</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bornkamm</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="D Eick"><span data-itemprop=givenNames><span itemprop=givenName>D</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eick</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Kohlhuber"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kohlhuber</span></span></li> </ol><time itemprop=datePublished datetime=1999>1999</time><span itemprop=headline>Control of cell growth by c-Myc in the absence of cell division</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>9</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Current Biology</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>1255</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>1258</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Control%20of%20cell%20growth%20by%20c-Myc%20in%20the%20absence%20of%20cell%20division"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib31> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Silberzahn"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Silberzahn</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="EL Uhlmann"><span data-itemprop=givenNames><span itemprop=givenName>EL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Uhlmann</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DP Martin"><span data-itemprop=givenNames><span itemprop=givenName>DP</span></span><span data-itemprop=familyNames><span itemprop=familyName>Martin</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="P Anselmi"><span data-itemprop=givenNames><span itemprop=givenName>P</span></span><span data-itemprop=familyNames><span itemprop=familyName>Anselmi</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Aust"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Aust</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="EC Awtrey"><span data-itemprop=givenNames><span itemprop=givenName>EC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Awtrey</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="Š Bahník"><span data-itemprop=givenNames><span itemprop=givenName>Š</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bahník</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Bai"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bai</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Bannard"><span data-itemprop=givenNames><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bannard</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Bonnier"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bonnier</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Carlsson"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Carlsson</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Cheung"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Cheung</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="G Christensen"><span data-itemprop=givenNames><span itemprop=givenName>G</span></span><span data-itemprop=familyNames><span itemprop=familyName>Christensen</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Clay"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Clay</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MA Craig"><span data-itemprop=givenNames><span itemprop=givenName>MA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Craig</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Dalla Rosa"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Dalla</span><span itemprop=familyName>Rosa</span></span> </li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Dam"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Dam</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MH Evans"><span data-itemprop=givenNames><span itemprop=givenName>MH</span></span><span data-itemprop=familyNames><span itemprop=familyName>Evans</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Cervantes"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Cervantes</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="N Fong"><span data-itemprop=givenNames><span itemprop=givenName>N</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fong</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Gamez-Djokic"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Gamez-Djokic</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Glenz"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Glenz</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Gordon-McKeon"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Gordon-McKeon</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Heaton"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Heaton</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KH Eriksson"><span data-itemprop=givenNames><span itemprop=givenName>KH</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eriksson</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Heene"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Heene</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Hofelich Mohr"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hofelich</span><span itemprop=familyName>Mohr</span></span> </li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Högden"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Högden</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="K Hui"><span data-itemprop=givenNames><span itemprop=givenName>K</span></span><span data-itemprop=familyNames><span itemprop=familyName>Hui</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Johannesson"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Johannesson</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Kalodimos"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kalodimos</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Kaszubowski"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kaszubowski</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DM Kennedy"><span data-itemprop=givenNames><span itemprop=givenName>DM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kennedy</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="R Lei"><span data-itemprop=givenNames><span itemprop=givenName>R</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lei</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TA Lindsay"><span data-itemprop=givenNames><span itemprop=givenName>TA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lindsay</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Liverani"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Liverani</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CR Madan"><span data-itemprop=givenNames><span itemprop=givenName>CR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Madan</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="DC Molden"><span data-itemprop=givenNames><span itemprop=givenName>DC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Molden</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Molleman"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Molleman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RD Morey"><span data-itemprop=givenNames><span itemprop=givenName>RD</span></span><span data-itemprop=familyNames><span itemprop=familyName>Morey</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LB Mulder"><span data-itemprop=givenNames><span itemprop=givenName>LB</span></span><span data-itemprop=familyNames><span itemprop=familyName>Mulder</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="BA Nijstad"><span data-itemprop=givenNames><span itemprop=givenName>BA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nijstad</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B Pope"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pope</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="NG Pope"><span data-itemprop=givenNames><span itemprop=givenName>NG</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pope</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JM Prenoveau"><span data-itemprop=givenNames><span itemprop=givenName>JM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Prenoveau</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Rink"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rink</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Robusto"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Robusto</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Roderique"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Roderique</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Sandberg"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sandberg</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Schlueter"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schlueter</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="FD Schönbrodt"><span data-itemprop=givenNames><span itemprop=givenName>FD</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schönbrodt</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="MF Sherman"><span data-itemprop=givenNames><span itemprop=givenName>MF</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sherman</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SA Sommer"><span data-itemprop=givenNames><span itemprop=givenName>SA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sommer</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KL Sotak"><span data-itemprop=givenNames><span itemprop=givenName>KL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sotak</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SM Spain"><span data-itemprop=givenNames><span itemprop=givenName>SM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Spain</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="C Spörlein"><span data-itemprop=givenNames><span itemprop=givenName>C</span></span><span data-itemprop=familyNames><span itemprop=familyName>Spörlein</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Stafford"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Stafford</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Stefanutti"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Stefanutti</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Täuber"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Täuber</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Ullrich"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Ullrich</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Vianello"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Vianello</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E-J Wagenmakers"><span data-itemprop=givenNames><span itemprop=givenName>E-J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wagenmakers</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Witkowiak"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Witkowiak</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Yoon"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Yoon</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="BA Nosek"><span data-itemprop=givenNames><span itemprop=givenName>BA</span></span><span data-itemprop=familyNames><span itemprop=familyName>Nosek</span></span></li> </ol><time itemprop=datePublished datetime=2017>2017</time><span itemprop=headline>Many analysts, one dataset: Making transparent how variations in analytical choices affect results</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>PsyArXiv</span></span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Many%20analysts,%20one%20dataset:%20Making%20transparent%20how%20variations%20in%20analytical%20choices%20affect%20results"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib32> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="EF Trakhtenberg"><span data-itemprop=givenNames><span itemprop=givenName>EF</span></span><span data-itemprop=familyNames><span itemprop=familyName>Trakhtenberg</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="N Pho"><span data-itemprop=givenNames><span itemprop=givenName>N</span></span><span data-itemprop=familyNames><span itemprop=familyName>Pho</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KM Holton"><span data-itemprop=givenNames><span itemprop=givenName>KM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Holton</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="TW Chittenden"><span data-itemprop=givenNames><span itemprop=givenName>TW</span></span><span data-itemprop=familyNames><span itemprop=familyName>Chittenden</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JL Goldberg"><span data-itemprop=givenNames><span itemprop=givenName>JL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Goldberg</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Dong"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Dong</span></span></li> </ol><time itemprop=datePublished datetime=2016>2016</time><span itemprop=headline>Cell types differ in global coordination of splicing and proportion of highly expressed genes</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>6</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Scientific Reports</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>32249</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Cell%20types%20differ%20in%20global%20coordination%20of%20splicing%20and%20proportion%20of%20highly%20expressed%20genes"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib33> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="JC Valentine"><span data-itemprop=givenNames><span itemprop=givenName>JC</span></span><span data-itemprop=familyNames><span itemprop=familyName>Valentine</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="A Biglan"><span data-itemprop=givenNames><span itemprop=givenName>A</span></span><span data-itemprop=familyNames><span itemprop=familyName>Biglan</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="RF Boruch"><span data-itemprop=givenNames><span itemprop=givenName>RF</span></span><span data-itemprop=familyNames><span itemprop=familyName>Boruch</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="FG Castro"><span data-itemprop=givenNames><span itemprop=givenName>FG</span></span><span data-itemprop=familyNames><span itemprop=familyName>Castro</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="LM Collins"><span data-itemprop=givenNames><span itemprop=givenName>LM</span></span><span data-itemprop=familyNames><span itemprop=familyName>Collins</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="BR Flay"><span data-itemprop=givenNames><span itemprop=givenName>BR</span></span><span data-itemprop=familyNames><span itemprop=familyName>Flay</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Kellam"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Kellam</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="EK Mościcki"><span data-itemprop=givenNames><span itemprop=givenName>EK</span></span><span data-itemprop=familyNames><span itemprop=familyName>Mościcki</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="SP Schinke"><span data-itemprop=givenNames><span itemprop=givenName>SP</span></span><span data-itemprop=familyNames><span itemprop=familyName>Schinke</span></span></li> </ol><time itemprop=datePublished datetime=2011>2011</time><span itemprop=headline>Replication in prevention science</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>12</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Prevention Science</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>103</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>117</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Replication%20in%20prevention%20science"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib34> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="W Viechtbauer"><span data-itemprop=givenNames><span itemprop=givenName>W</span></span><span data-itemprop=familyNames><span itemprop=familyName>Viechtbauer</span></span></li> </ol><time itemprop=datePublished datetime=2010>2010</time><span itemprop=headline>Conducting Meta-Analyses in R with the metafor Package</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>36</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Journal of Statistical Software</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>1</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>48</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Conducting%20Meta-Analyses%20in%20R%20with%20the%20metafor%20Package"> </li> <li itemscope="" itemtype=http://schema.org/Article itemprop=citation id=bib35> <ol data-itemprop=authors> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Walz"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Walz</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Lorenzin"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Lorenzin</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="J Morton"><span data-itemprop=givenNames><span itemprop=givenName>J</span></span><span data-itemprop=familyNames><span itemprop=familyName>Morton</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="KE Wiese"><span data-itemprop=givenNames><span itemprop=givenName>KE</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wiese</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="B von Eyss"><span data-itemprop=givenNames><span itemprop=givenName>B</span></span><span data-itemprop=familyNames><span itemprop=familyName>von</span><span itemprop=familyName>Eyss</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Herold"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Herold</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Rycak"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Rycak</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="H Dumay-Odelot"><span data-itemprop=givenNames><span itemprop=givenName>H</span></span><span data-itemprop=familyNames><span itemprop=familyName>Dumay-Odelot</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="S Karim"><span data-itemprop=givenNames><span itemprop=givenName>S</span></span><span data-itemprop=familyNames><span itemprop=familyName>Karim</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Bartkuhn"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Bartkuhn</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="F Roels"><span data-itemprop=givenNames><span itemprop=givenName>F</span></span><span data-itemprop=familyNames><span itemprop=familyName>Roels</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="T Wüstefeld"><span data-itemprop=givenNames><span itemprop=givenName>T</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wüstefeld</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Fischer"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Fischer</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Teichmann"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Teichmann</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="L Zender"><span data-itemprop=givenNames><span itemprop=givenName>L</span></span><span data-itemprop=familyNames><span itemprop=familyName>Zender</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="CL Wei"><span data-itemprop=givenNames><span itemprop=givenName>CL</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wei</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="O Sansom"><span data-itemprop=givenNames><span itemprop=givenName>O</span></span><span data-itemprop=familyNames><span itemprop=familyName>Sansom</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="E Wolf"><span data-itemprop=givenNames><span itemprop=givenName>E</span></span><span data-itemprop=familyNames><span itemprop=familyName>Wolf</span></span></li> <li itemscope="" itemtype=http://schema.org/Person itemprop=author> <meta itemprop=name content="M Eilers"><span data-itemprop=givenNames><span itemprop=givenName>M</span></span><span data-itemprop=familyNames><span itemprop=familyName>Eilers</span></span></li> </ol><time itemprop=datePublished datetime=2014>2014</time><span itemprop=headline>Activation and repression by oncogenic MYC shape tumour-specific gene expression profiles</span><span itemscope="" itemtype=http://schema.org/PublicationVolume itemprop=isPartOf><span itemprop=volumeNumber data-itemtype=http://schema.org/Number>511</span><span itemscope="" itemtype=http://schema.org/Periodical itemprop=isPartOf><span itemprop=name>Nature</span></span></span><span itemprop=pageStart data-itemtype=http://schema.org/Number>483</span><span itemprop=pageEnd data-itemtype=http://schema.org/Number>487</span><span itemscope="" itemtype=http://schema.org/Organization itemprop=publisher> <meta itemprop=name content=Unknown><span itemscope="" itemtype=http://schema.org/ImageObject itemprop=logo> <meta itemprop=url content="https://via.placeholder.com/600x60/dbdbdb/4a4a4a.png?text=Unknown"> </span></span> <meta itemprop=image content="https://via.placeholder.com/1200x714/dbdbdb/4a4a4a.png?text=Activation%20and%20repression%20by%20oncogenic%20MYC%20shape%20tumour-specific%20gene%20expression%20profiles"> </li> </ol> </section> </article>